<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <meta name="author" content="First Author ^{1,*}, Co-Author ^{2} and Co-Author ^{1,2}" />
  <title>The evolutionary consequences of adaptive phenotypic plasticity</title>
  <style>
    html {
      line-height: 1.7;
      font-family: Georgia, serif;
      font-size: 20px;
      color: #1a1a1a;
      background-color: #fdfdfd;
    }
    body {
      margin: 0 auto;
      max-width: 40em;
      padding-left: 50px;
      padding-right: 50px;
      padding-top: 50px;
      padding-bottom: 50px;
      hyphens: auto;
      word-wrap: break-word;
      text-rendering: optimizeLegibility;
      font-kerning: normal;
    }
    @media (max-width: 600px) {
      body {
        font-size: 0.9em;
        padding: 1em;
      }
    }
    @media print {
      body {
        background-color: transparent;
        color: black;
      }
      p, h2, h3 {
        orphans: 3;
        widows: 3;
      }
      h2, h3, h4 {
        page-break-after: avoid;
      }
    }
    p {
      margin-top: 1.7em;
    }
    a {
      color: #1a1a1a;
    }
    a:visited {
      color: #1a1a1a;
    }
    img {
      max-width: 100%;
    }
    h1, h2, h3, h4, h5, h6 {
      margin-top: 1.7em;
    }
    ol, ul {
      padding-left: 1.7em;
      margin-top: 1.7em;
    }
    li > ol, li > ul {
      margin-top: 0;
    }
    blockquote {
      margin: 1.7em 0 1.7em 1.7em;
      padding-left: 1em;
      border-left: 2px solid #e6e6e6;
      font-style: italic;
    }
    code {
      font-family: Menlo, Monaco, 'Lucida Console', Consolas, monospace;
      background-color: #f0f0f0;
      font-size: 85%;
      margin: 0;
      padding: .2em .4em;
    }
    pre {
      line-height: 1.5em;
      padding: 1em;
      background-color: #f0f0f0;
      overflow: auto;
    }
    pre code {
      padding: 0;
      overflow: visible;
    }
    hr {
      background-color: #1a1a1a;
      border: none;
      height: 1px;
      margin-top: 1.7em;
    }
    table {
      border-collapse: collapse;
      width: 100%;
      overflow-x: auto;
      display: block;
    }
    th, td {
      border-bottom: 1px solid lightgray;
      padding: 1em 3em 1em 0;
    }
    header {
      margin-bottom: 6em;
      text-align: center;
    }
    nav a:not(:hover) {
      text-decoration: none;
    }
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    span.underline{text-decoration: underline;}
    div.column{display: inline-block; vertical-align: top; width: 50%;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    .display.math{display: block; text-align: center; margin: 0.5rem auto;}
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
    }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
</head>
<body>
<header id="title-block-header">
<h1 class="title">The evolutionary consequences of adaptive phenotypic plasticity</h1>
<p class="author">First Author <span class="math inline"><sup>1, *</sup></span>, Co-Author <span class="math inline"><sup>2</sup></span> and Co-Author <span class="math inline"><sup>1, 2</sup></span></p>
</header>
<h1 id="introduction">Introduction</h1>
<p>Fluctuating environmental conditions are ubiquitous in natural systems. Organisms have evolved a wide range of evolved strategies for coping with environmental change, such as phenotypic plasticity <span class="citation" data-cites="ghalambor_adaptive_2007">(C. K. Ghalambor et al. 2007)</span>, bet hedging <span class="citation" data-cites="beaumont_experimental_2009">(Beaumont et al. 2009)</span>, periodic migration <span class="citation" data-cites="winger_long_2019">(Winger et al. 2019)</span>, and adaptive tracking <span class="citation" data-cites="barrett_adaptation_2008">(Barrett and Schluter 2008)</span>. The particular coping mechanisms that evolve in fluctuating environments shift the course of subsequent evolution <span class="citation" data-cites="wennersten_population-level_2012 schaum_plasticity_2014">(Wennersten and Forsman 2012; Schaum and Collins 2014)</span>. Identifying the mechanisms most likely to evolve and examining both the evolutionary constraints and opportunities associated with each is critical for us to understand and predict evolutionary outcomes in changing environments.</p>
<p>In this work, we focus on phenotypic plasticity, which can be defined as the capacity for a single genotype to alter phenotypic expression in response to a change in its environment <span class="citation" data-cites="west-eberhard_developmental_2003">(May Jane West-Eberhard 2003)</span>. Phenotypic plasticity is controlled by genes whose expression is coupled to one or more abiotic or biotic environmental signals. For example, the sex ratio of the crustacean <em>Gammarus duebeni</em> is modulated by changes in photoperiod and temperature <span class="citation" data-cites="dunn_two_2005">(Dunn et al. 2005)</span>, and the reproductive output of some invertebrate species is heightened when infected with parasites to compensate for offspring loss <span class="citation" data-cites="chadwick_parasite-mediated_2005">(Chadwick and Little 2005)</span>. In this study, we conducted digital evolution experiments to investigate how the evolution of adaptive phenotypic plasticity shifts the course of evolution in a cyclically changing environment. Specifically, we examined the effects of adaptive plasticity on subsequent genomic and phenotypic change, the capacity to evolve and then maintain novel traits, and the accumulation of deleterious alleles.</p>
<p>Evolutionary biologists have long been interested in how evolutionary change across generations is influenced by phenotypic plasticity because of its role in generating phenotypic variance <span class="citation" data-cites="gibert_phenotypic_2019">(Gibert, Debat, and Ghalambor 2019)</span>. The effects of phenotypic plasticity on adaptive evolution have been disputed, as few studies have been able to observe both the initial patterns of plasticity and the subsequent divergence of traits in natural populations <span class="citation" data-cites="ghalambor_adaptive_2007 wund_assessing_2012 forsman_rethinking_2015 ghalambor_non-adaptive_2015 hendry_key_2016">(C. K. Ghalambor et al. 2007; Wund 2012; Forsman 2015; Cameron K. Ghalambor et al. 2015; Hendry 2016)</span>. In a changing environment, adaptive phenotypic plasticity provides a mechanism for organisms to regulate trait expression within their lifetime, which can stabilize populations through changes <span class="citation" data-cites="gibert_phenotypic_2019">(Gibert, Debat, and Ghalambor 2019)</span>. In this context, the stabilizing effect of adaptive plasticity has been hypothesized to constrain the rate of adaptive evolution <span class="citation" data-cites="gupta_study_1982 ancel_undermining_2000 huey_behavioral_2003 price_role_2003 paenke_influence_2007">(Gupta and Lewontin 1982; Ancel 2000; Huey, Hertz, and Sinervo 2003; Price, Qvarnström, and Irwin 2003; Paenke, Sendhoff, and Kawecki 2007)</span>. That is, directional selection may be weak if environmentally-induced phenotypes are close to the optimum; as such, adaptively plastic populations may evolve slowly (relative to non-plastic populations) unless there is a substantial fitness cost to plasticity.</p>
<figure>
<embed src="media/reaction-norms.pdf" id="fig:reaction-norms" /><figcaption aria-hidden="true"> <strong>Hypothetical reaction norms for genotypes that exhibit phenotypic variation.</strong> (a) through (d) show four hypothetical reaction norm scenarios for the environmental change described in (e). In (e), the environment changes from E<span class="math inline"><sub>1</sub></span> (in red) to E<span class="math inline"><sub>2</sub></span> (in blue), and the optimal phenotypes for environments E<span class="math inline"><sub>1</sub></span> and E<span class="math inline"><sub>2</sub></span> are different (O<span class="math inline"><sub>E1</sub></span> and O<span class="math inline"><sub>E2</sub></span>, respectively). In each of the four reaction norm scenarios, populations are well-adapted to E<span class="math inline"><sub>1</sub></span>. In (a), genotypes in the population are non-plastic, and as such, we would expect strong directional selection on mutations that move phenotypes toward O<span class="math inline"><sub>E2</sub></span> after the environment changes. In (b), genotypes in the population are adaptively plastic. That is, phenotypic changes induced by the environment change to E<span class="math inline"><sub>2</sub></span> are near the optimum, and as such, we would expect this population to remain relatively stable after the environment changes. In (c), the population exhibits non-adaptive plasticity with substantial variation in how individuals respond to the environmental change. In this case, we expect plasticity to result in a rapid evolutionary response to the change in environment. In (d), the population exhibits maladaptive plasticity relative to the given environmental change. When the environment changes, there is little variation for selection to act on, and without beneficial mutations, this population may be at risk of extinction due to their maladaptive plastic response to the environment. </figcaption>
</figure>
<p>Phenotypic plasticity allows for the accumulation of genetic variation in genomic regions that are unexpressed under current environmental conditions. Such cryptic (“hidden”) genetic variation can serve as a source of diversity in the population, upon which selection can act when the environment changes <span class="citation" data-cites="schlichting_hidden_2008 levis_evaluating_2016">(Schlichting 2008; Levis and Pfennig 2016)</span>. It remains unclear to what extent and under what circumstances this cryptic variation caches adaptive potential or merely accumulates deleterious alleles <span class="citation" data-cites="gibson_uncovering_2004 paaby_cryptic_2014 zheng_cryptic_2019">(Gibson and Dworkin 2004; Paaby and Rockman 2014; Zheng, Payne, and Wagner 2019)</span>.</p>
<p>The “genes as followers” hypothesis (also known as the “plasticity first” hypothesis) predicts that phenotypic plasticity may facilitate adaptive evolutionary change by producing variants with enhanced fitness under stressful or novel conditions <span class="citation" data-cites="west-eberhard_developmental_2003 schwander_genes_2011 levis_evaluating_2016">(May Jane West-Eberhard 2003; Schwander and Leimar 2011; Levis and Pfennig 2016)</span>. Environmentally-induced trait changes can be refined through selection over time (<em>i.e.</em>, genetic accommodation). Further, selection may drive plastic phenotypes to lose their environmental dependence over time in a process known as genetic assimilation <span class="citation" data-cites="west-eberhard_developmental_2005 pigliucci_phenotypic_2006 crispo_baldwin_2007 schlichting_phenotypic_2014 levis_evaluating_2016">(M. J. West-Eberhard 2005; Pigliucci 2006; Crispo 2007; Schlichting and Wund 2014; Levis and Pfennig 2016)</span>. In this way, environmentally-induced phenotypic changes can precede an evolutionary response.</p>
<p>Phenotypic plasticity may also “rescue” populations from extinction under changing environmental conditions by buffering populations against novel stressors. This buffer promotes stability and persistence and grants populations time to further adapt to rapidly changing environmental conditions <span class="citation" data-cites="west-eberhard_developmental_2003 chevin_when_2010">(May Jane West-Eberhard 2003; Chevin and Lande 2010)</span>. Disparate predictions about how phenotypic plasticity may shift the course of subsequent evolution are not necessarily mutually exclusive. Figure <a href="#fig:reaction-norms" data-reference-type="ref" data-reference="fig:reaction-norms">1</a> overviews how we might expect different forms of phenotypic plasticity to result in different evolutionary responses after an environmental change. Genetic and environmental contexts determine if and to what extent phenotypic plasticity promotes or constrains subsequent evolution.</p>
<p>Experimental studies investigating the relationship between phenotypic plasticity and evolutionary outcomes can be challenging to conduct in natural systems. Such experiments would require the ability to irreversibly toggle plasticity followed by long periods of evolution during which detailed phenotypic data would need to be collected. Digital evolution experiments have emerged as a powerful research framework from which evolution can be studied. In digital evolution, self-replicating computer programs (digital organisms) compete for resources, mutate, and evolve following Darwinian dynamics <span class="citation" data-cites="wilke_biology_2002">(Wilke and Adami 2002)</span>. Digital evolution studies balance the speed and transparency of mathematical and computational simulations with the open-ended realism of laboratory experiments. Modern computers allow us to observe many generations of digital evolution at tractable time scales; thousands of generations can take mere minutes as opposed to months, years, or centuries. Digital evolution systems also allow for perfect, non-invasive data tracking. Such transparency permits the tracking of complete evolutionary histories within an experiment, which circumvents the historical problem of drawing evolutionary inferences using incomplete records (from frozen samples or even fossils) and extant genetic sequences. Additionally, digital evolution systems allow for experimental manipulations and analyses that go beyond what is possible in wet-lab experiments. Such analyses have included exhaustive knockouts of every loci to identify the functionality of each <span class="citation" data-cites="lenski_evolutionary_2003">(Lenski et al. 2003)</span>, comprehensive characterization of local mutational landscapes <span class="citation" data-cites="lenski_genome_1999 canino-koning_fluctuating_2019">(Lenski et al. 1999; Canino-Koning, Wiser, and Ofria 2019)</span>, and the real-time reversion of all deleterious mutations as they occur to isolate their long-term effects on evolutionary outcomes <span class="citation" data-cites="covert_experiments_2013">(Covert et al. 2013)</span>. Digital evolution studies allow us to directly toggle the possibility for adaptive plastic responses to evolve, which enables us to empirically test hypotheses that were previously relegated to theoretical analyses.</p>
<p>In this work, we use the Avida Digital Evolution Platform <span class="citation" data-cites="ofria_avida:_2009">(Ofria, Bryson, and Wilke 2009)</span>. Avida is an open-source system that has been used to conduct a wide range of well-regarded studies on evolutionary dynamics, including the origins of complex features <span class="citation" data-cites="lenski_evolutionary_2003">(Lenski et al. 2003)</span>, the survival of the flattest effect <span class="citation" data-cites="wilke_evolution_2001">(Wilke et al. 2001)</span>, and the origins of reproductive division of labor <span class="citation" data-cites="goldsby_evolutionary_2014">(Goldsby et al. 2014)</span>. Our experiments build directly on previous studies in Avida that characterized the <em>de novo</em> evolution of adaptive phenotypic plasticity <span class="citation" data-cites="clune_investigating_2007 lalejini_evolutionary_2016">(Clune, Ofria, and Pennock 2007; Lalejini and Ofria 2016)</span> as well as previous work investigating the evolutionary consequences of fluctuating environments for populations of non-plastic digital organisms <span class="citation" data-cites="wilke_evolution_2001 canino-koning_fluctuating_2019">(Wilke et al. 2001; Canino-Koning, Wiser, and Ofria 2019)</span>. Of particular relevance, <span class="citation" data-cites="clune_investigating_2007">(Clune, Ofria, and Pennock 2007)</span> and <span class="citation" data-cites="lalejini_evolutionary_2016">(Lalejini and Ofria 2016)</span> experimentally demonstrated that adaptive phenotypic plasticity can evolve given the following four conditions (as identified by <span class="citation" data-cites="ghalambor_behavior_2010">(Cameron K. Ghalambor, Angeloni, and Carroll 2010)</span>): (1) populations experience temporal environmental variation (<em>i.e.</em>, a fluctuating environment), (2) these environments are differentiable by reliable cues, (3) each environment favors different phenotypic traits, and (4) no single phenotype exhibits high fitness across all environments. We build on this previous work, but shift our focus from the evolutionary causes of adaptive phenotypic plasticity to investigate its evolutionary consequences in a fluctuating environment.</p>
<p>Each of our experiments are divided into two phases: in phase one, we precondition sets of founder organisms with differing plastic or non-plastic adaptations; in phase two, we examine the subsequent evolution of populations founded with organisms from phase one under specific environmental conditions. First, we examine the evolutionary histories of phase two populations to test whether adaptive plasticity constrained subsequent genomic and phenotypic changes. Next, we evaluate how adaptive plasticity influences fitness landscape exploration and exploitation by identifying how well populations produced by each type of founder are able to evolve and retain novel adaptive traits. Finally, we examine lineages to determine whether adaptive plasticity facilitated the accumulation of cryptic genetic variation that would prove deleterious when the environment changed.</p>
<p>We found that the evolution of adaptive plasticity reduced subsequent rates of evolutionary change in a cyclic environment. The non-plastic populations underwent more frequent selective sweeps and accumulated many more genetic changes over time, as non-plastic populations relied on genetic variation from <em>de novo</em> mutations to continuously re-adapt to environmental changes. We found that the evolution of adaptive phenotypic plasticity buffers populations against environmental fluctuations, whereas repeated selective sweeps in non-plastic populations drive the accumulation of deleterious genes and the loss of secondary beneficial traits via deleterious hitchhiking. As such, adaptively plastic populations were better able to retain novel traits than their non-plastic counterparts.</p>
<h1 id="materials-and-methods">Materials and Methods</h1>
<h2 id="the-avida-digital-evolution-platform">The Avida Digital Evolution Platform</h2>
<p>Avida is a study system wherein self-replicating computer programs (digital organisms) compete for space on a finite toroidal grid <span class="citation" data-cites="ofria_avida:_2009">(Ofria, Bryson, and Wilke 2009)</span>. Each digital organism is defined by a linear sequence of program instructions (its genome) and a set of virtual hardware components used to interpret and express those instructions. Genomes are expressed sequentially except when the execution of one instruction deterministically changes which instruction should be executed next (<em>e.g.</em>, a “jump” instruction). Genomes are built using an instruction set that is both robust (<em>i.e.</em>, any ordering of instructions is syntactically valid, though not necessarily meaningful) and Turing Complete (<em>i.e.</em>, able to represent any computable function, though not necessarily in an efficient manner). The instruction set includes operations for basic computations, flow control (<em>e.g.</em>, conditional logic and looping), input, output, and self-replication.</p>
<p>Organisms in Avida reproduce asexually by copying their genome instruction-by-instruction and then dividing. However, copy operations are imperfect and can result in single-instruction substitution mutations in an offspring’s genome. For this work, we configured copy operations to err at a rate of one expected mutation for every 400 instructions copied (<em>i.e.</em>, a per-instruction error rate of 0.0025). We held individual genomes at a fixed length of 100 instructions; that is, we did not include insertion and deletion mutations. We used fixed-length genomes to control for treatment-specific conditions resulting in the evolution of substantially different genome sizes <span class="citation" data-cites="supplemental_material">(<strong>supplemental_material?</strong>)</span><a href="#fn1" class="footnote-ref" id="fnref1" role="doc-noteref"><sup>1</sup></a>, which could, on its own, drive differences in evolutionary outcomes among experimental treatments. When an organism divides in Avida, its offspring is placed in a random location on the toroidal grid, replacing any previous occupant. For this work, we used the default 60 by 60 grid size, which limits the maximum population size to 3600 organisms. As such, improvements to the speed of self-replication are advantageous in the competition for space.</p>
<p>During evolution, organism replication rates improve in two ways: by improving genome efficiency (<em>e.g.</em>, using a more compact encoding) or by accelerating the rate at which the genome is expressed (their “metabolic rate”). An organism’s metabolic rate determines the speed at which it executes instructions in its genome. Initially, an organism’s metabolic rate is proportional to the length of its genome, but that rate is adjusted as it completes designated tasks, such as performing Boolean logic computations <span class="citation" data-cites="ofria_avida:_2009">(Ofria, Bryson, and Wilke 2009)</span>. In this way, we can reward or punish particular phenotypic traits.</p>
<h3 id="phenotypic-plasticity-in-avida">Phenotypic plasticity in Avida</h3>
<p>In this work, we measure a digital organism’s phenotype as the set of Boolean logic functions that it performs in a given environment. Sensory instructions in the Avida instruction set allow organisms to detect how performing a particular logic function would affect their metabolic rate (see supplemental material for more details, <span class="citation" data-cites="supplemental_material">(<strong>supplemental_material?</strong>)</span>). We define a phenotypically plastic organism as one that uses sensory information to alter which logic functions it performs based on the environment.</p>
<p>Phenotypic plasticity in Avida can be adaptive or non-adaptive for a given set of environments. Adaptive plasticity shifts net task expression closer to the optimum for the given environments. Non-adaptive plasticity changes task expression in either a neutral or deleterious way. Optimal plasticity toggles tasks to always perfectly match the set of rewarded tasks for the given set of environments.</p>
<h2 id="sec:methods:experiment">Experimental design</h2>
<figure>
<embed src="media/experimental-design.pdf" id="fig:experimental-design" style="width:100.0%" /><figcaption aria-hidden="true"> <strong>Overview of experimental design.</strong> Panel a shows the three environments used in every experiment and whether they reward or punish each base task. Additionally, the last two subplots in (a) show the additional tasks added in Phases 2B and 2C. All novel tasks confer a 10% metabolic reward, while executing the <code>poison</code> task causes a 10% metabolic punishment. Panel b shows how treatments are defined and the phase transition. Treatments are defined on the left, with each treatment consisting of an environment timeline and whether sensors are functional. This also shows Phase 1, the preconditioning phase. We conducted experiments using three different second phases, as shown on the right. Phase 1 is repeated for <em>each</em> Phase 2, with 100 populations of each treatment. All STATIC and NONPLASTIC replicates move on to Phase 2, while PLASTIC populations are only continued if their most abundant genotype exhibits optimal plasticity. Metrics are recorded only in Phase 2. </figcaption>
</figure>
<p>We conducted three independent experiments using Avida to investigate how the evolution of adaptive plasticity influences evolutionary outcomes in fluctuating environments. For each experiment, we compared the evolutionary outcomes of populations evolved under three treatments (Figure <a href="#fig:experimental-design" data-reference-type="ref" data-reference="fig:experimental-design">2</a>): (1) a <strong>PLASTIC</strong> treatment where the environment fluctuates, and digital organisms can use sensory instructions to differentiate between environmental states; (2) a <strong>NON-PLASTIC</strong> treatment with identical environment fluctuations, but where sensory instructions are disabled; and (3) a <strong>STATIC</strong> control where organisms evolve in a constant environment.</p>
<p>Each experiment was divided into two phases that each lasted for 200,000 updates<a href="#fn2" class="footnote-ref" id="fnref2" role="doc-noteref"><sup>2</sup></a> of evolution (Figure <a href="#fig:experimental-design" data-reference-type="ref" data-reference="fig:experimental-design">2</a>), which is approximately 30,000 to 40,000 generations. In phase one of each experiment, we preconditioned populations to their treatment-specific conditions. In phase two, we founded new populations with the evolved organisms from phase one and examined their subsequent evolution under given combinations of treatment and experimental conditions. During phase two, we tracked each population’s evolutionary history as well as saving the full final population. Phase one was for pre-conditioning only; all comparisons between treatments were performed on phase two data.</p>
<h3 id="sec:methods:experiment:environments">Environments</h3>
<p>We constructed three experimental environments, abbreviated hereafter as “ENV-A,” “ENV-B,” and “ENV-ALL.” Figure [YY] describes these environments based on whether each of six Boolean logic tasks (NOT, NAND, AND, OR-NOT, OR, and AND-NOT) is rewarded or punished. A rewarded task performed by an organism doubles their metabolic rate, allowing them to execute twice as many instructions in the same amount of time. A punished task halves an organism’s metabolic rate.</p>
<p>In both the PLASTIC and NON-PLASTIC conditions, the environment cycles between equal-length periods of ENV-A and ENV-B. Each of these periods persist for 100 updates (approximately 15 to 20 generations). Thus, populations experience a total of 1,000 full periods of ENV-A interlaced with 1,000 full periods of ENV-B during each experimental phase.</p>
<p>Organisms in the PLASTIC treatments differentiate between ENV-A and ENV-B by executing one of six sensory instructions, each associated with a particular logical task; these sensory instructions detect whether their associated task is currently rewarded or punished. By using sensory information in combination with execution flow-control instructions, organisms can conditionally perform different logic tasks depending on the current environmental conditions.</p>
<h3 id="sec:methods:experiment:phase-one">Experiment Phase 1 – Environment preconditioning</h3>
<p>For each treatment, we founded 100 independent populations from a common ancestral strain capable only of self-replication. At the end of phase one, we identified the most abundant (<em>i.e.</em>, dominant) genotype and extracted an organism with that genotype from each replicate population to found a new population for phase two.</p>
<p>For the PLASTIC treatment, we measure plasticity by independently testing a given genotype in each of ENV-A and ENV-B. We discard phase one populations if the dominant genotype does not exhibit optimal plasticity. This approach ensures that measurements taken on PLASTIC-treatment populations during the second phase of each experiment reflect the evolutionary consequences of adaptive plasticity.</p>
<h3 id="sec:methods:exp:evolutionary-change-rate">Experiment Phase 2A – Evolutionary change rate</h3>
<p>Phase 2A continued exactly as phase one, except we tracked the rates of evolutionary change in each of the PLASTIC-, NON-PLASTIC-, and STATIC-treatment populations. Specifically, we quantified evolutionary change rates using four metrics (each described in Table <a href="#tab:metrics-definitions" data-reference-type="ref" data-reference="tab:metrics-definitions">1</a>): (1) coalescence event count, (2) mutation count, (3) phenotypic volatility, and (4) mutational stability. We additionally used knockout experiments to examine how the genetic architectures of organisms and their ancestors changed over time.</p>
<h3 id="sec:methods:exp:novel-task-evolution">Experiment Phase 2B – Novel task evolution</h3>
<p>Phase 2B extended the conditions of phase one by adding 71 novel Boolean logic tasks, which were always rewarded in all treatments <span class="citation" data-cites="ofria_avida:_2009">(Ofria, Bryson, and Wilke 2009)</span>. The original six phase one tasks (NOT, NAND, AND, OR-NOT, OR, and AND-NOT; hereafter called “base” tasks) continued to be rewarded or punished according to the particular treatment conditions. An organism’s metabolic rate was increased by 10% for each novel task that it performed (limited to one reward per task). This reward provided a selective pressure to evolve these tasks, but their benefits did not overwhelm the 100% increase conferred by rewarded base tasks. As such, populations in the PLASTIC and NON-PLASTIC treatments could not easily escape environmental fluctuations by abandoning the fluctuating base tasks.</p>
<p>During this experiment, we tracked the extent to which populations evolving under each treatment were capable of acquiring and retaining novel tasks. Specifically, we used three metrics (each described in Table <a href="#tab:metrics-definitions" data-reference-type="ref" data-reference="tab:metrics-definitions">1</a>): (1) final novel task count, (2) novel task discovery, and (3) novel task loss.</p>
<h3 id="sec:methods:exp:deleterious-instruction-accumulation">Experiment Phase 2C – Deleterious instruction accumulation</h3>
<p>Phase 2C extended the instruction set of phase one with a <code>poison</code> instruction. When an organism executes a <code>poison</code> instruction, it performs a “poisonous” task, which reduces the organism’s metabolic rate (and thus reproductive success) but does not otherwise alter the organism’s function. We imposed a 10% penalty each time an organism performed the poisonous task, making the <code>poison</code> instruction explicitly deleterious to execute. We did not limit the number of times that an organism could perform the poisonous task, and as such, organisms could perform the poisonous task as many times as they executed the <code>poison</code> instruction.</p>
<p>We tracked the number of times each organism along the dominant lineage performed the poisonous task. Specifically, we used two metrics (each described in Table <a href="#tab:metrics-definitions" data-reference-type="ref" data-reference="tab:metrics-definitions">1</a>): (1) final poisonous task count, and (2) poisonous task acquisition.</p>
<h2 id="experimental-analyses">Experimental analyses</h2>
<div id="tab:metrics-definitions">
<table>
<caption><strong>Metric descriptions.</strong></caption>
<thead>
<tr class="header">
<th style="text-align: left;"></th>
<th style="text-align: center;"><strong>Description</strong></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: left;">Coalescence event count</td>
<td style="text-align: left;">Number of coalescence events that have occurred, which indicates the frequency of selective sweeps in the population.</td>
</tr>
<tr class="even">
<td style="text-align: left;">Mutation count</td>
<td style="text-align: left;">Sum of all mutations that have occurred along a lineage.</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Phenotypic volatility</td>
<td style="text-align: left;">Number of instances where parent and offspring phenotypic profiles do not match along a lineage. Phenotypic volatility as defined here indicates the rate at which accumulated genetic changes actually change the phenotype along a lineage.</td>
</tr>
<tr class="even">
<td style="text-align: left;">Mutational stability</td>
<td style="text-align: left;">Proportion of mutated offspring along a lineage whose phenotypic profile matches that of their parent.</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Final novel task count</td>
<td style="text-align: left;">Count of unique novel tasks performed by the representative organism in a final population from experiment . This metric can range from 0 to 71 and measures the level of exploitation of the fitness landscape (<em>i.e.</em>, the mapping between genetic space and phenotype space) at a given point in time.</td>
</tr>
<tr class="even">
<td style="text-align: left;">Novel task discovery</td>
<td style="text-align: left;">Number of unique novel tasks ever performed along a given lineage in experimental , even if a task is later lost. This metric can range from 0 to 71 and measures a given lineage’s level of exploration of the fitness landscape.</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Novel task loss</td>
<td style="text-align: left;">Number of instances along a given lineage from experimental that a novel task is performed by a parent but not its offspring. This metric measures how often a given lineage fails to retain evolved traits over time.</td>
</tr>
<tr class="even">
<td style="text-align: left;">Final poisonous task count</td>
<td style="text-align: left;">Number of times the poisonous task is performed by the representative organism from a final population from experiment .</td>
</tr>
<tr class="odd">
<td style="text-align: left;">Poisonous task acquisition count</td>
<td style="text-align: left;">Number of instances along a given lineage where a mutation causes an offspring perform the poisonous task more times than its parent.</td>
</tr>
</tbody>
</table>
</div>
<p>For each of our experiments, we tracked and analyzed the phylogenetic histories of evolving populations during phase two. For each replicate, we identified an organism with the most abundant genotype in the final evolved population that we used as a representative organism for further analysis. We then isolated the lineage from the founding organism to the representative organism, which we used as the representative lineage for further analysis. We manually inspected evolved phylogenies and found no evidence that any of our experimental treatments supported long-term coexistence. As such, each of the representative lineages reflect the majority of evolutionary history from a given population at the end of our experiment.</p>
<p>Some of our metrics required us to measure genotype-by-environment interactions. Importantly, in the fluctuating environments, we needed to differentiate phenotypic changes that were caused by mutations from those that were caused by environmental changes. To accomplish this task, we produced organisms with the focal genotype, measured their phenotype in each environment, and aggregated the resulting phenotypes to create a <em>phenotypic profile</em>. Although organisms with different genotypes may express the same set of tasks across environments, their phenotypic profiles may not necessarily be the same. For example, an organism that expresses NOT in ENV-A and NAND in ENV-B has a distinct profile from one that expresses NAND in ENV-A and NOT in ENV-B.</p>
<p>For an individual organism, we can perform knockout experiments to identify which instructions are responsible for producing a given phenotypic outcome. To perform a knockout, we duplicate the organism, replacing a single instruction with an inert “no-operation” instruction. We then identify any phenotypic changes by contrasting the execution results of the “knockout” organism and the original. Such changes provide evidence of the role that the original instruction must have played in the genome. For example, when an organism performs the NAND task but loses it when an instruction is knocked out, we categorize that instruction as part of the NAND task machinery. We use knockout experiments to characterize the role of each instruction in the genomes of every organism along all study lineages, revealing how genetic architectures change over time.</p>
<h2 id="statistical-analyses">Statistical analyses</h2>
<p>Across all of our experiments, we differentiated between sample distributions using non-parametric statistical tests. For each major analysis, we first performed a Kruskal-Wallis test <span class="citation" data-cites="kruskal_use_1952">(Kruskal and Wallis 1952)</span> to determine if there were significant differences in results from the PLASTIC, NON-PLASTIC, and STATIC treatments (significance level <span class="math inline"><em>α</em> = 0.05</span>). If so, we applied a Wilcoxon rank-sum test <span class="citation" data-cites="kotz_individual_1992">(Wilcoxon 1992)</span> to distinguish between pairs of treatments. We applied Bonferroni corrections for multiple comparisons <span class="citation" data-cites="rice_analyzing_1989">(Rice 1989)</span> where appropriate.</p>
<h2 id="software-availability">Software availability</h2>
<p>We conducted our experiments using a modified version of the Avida software, which is open source and freely available on <a href="https://github.com/amlalejini/evolutionary-consequences-of-plasticity">GitHub</a> <span class="citation" data-cites="supplemental_material">(<strong>supplemental_material?</strong>)</span>. We used Python for data processing, and we conducted all statistical analyses using R version 4 <span class="citation" data-cites="r_core_team_r_v4">(<strong>r_core_team_r_v4?</strong>)</span>. We used the tidyverse collection of R packages <span class="citation" data-cites="r_tidyverse_2019">(<strong>r_tidyverse_2019?</strong>)</span> to wrangle data, and we used the following R packages for analysis, graphing, and visualization: ggplot2 <span class="citation" data-cites="R-ggplot2">(<strong>R-ggplot2?</strong>)</span>, cowplot <span class="citation" data-cites="R-cowplot">(<strong>R-cowplot?</strong>)</span>, Color Brewer <span class="citation" data-cites="harrower_colorbrewerorg_2003 R-Brewer_2014">(<strong>harrower_colorbrewerorg_2003?</strong>; <strong>R-Brewer_2014?</strong>)</span>, rstatix <span class="citation" data-cites="R-rstatix">(<strong>R-rstatix?</strong>)</span>, ggsignif <span class="citation" data-cites="R-ggsignif">(<strong>R-ggsignif?</strong>)</span>, scales <span class="citation" data-cites="R-scales">(<strong>R-scales?</strong>)</span>, Hmisc <span class="citation" data-cites="R-Hmisc">(<strong>R-Hmisc?</strong>)</span>, fmsb <span class="citation" data-cites="R-fmsb">(<strong>R-fmsb?</strong>)</span>, and boot <span class="citation" data-cites="R-boot">(<strong>R-boot?</strong>)</span>. We used R markdown <span class="citation" data-cites="rmarkdown">(<strong>rmarkdown?</strong>)</span> and bookdown <span class="citation" data-cites="R-bookdown">(<strong>R-bookdown?</strong>)</span> to generate web-enabled supplemental material. All of the source code for our experiments and analyses, including guides for replication and configuration files, can be found in our supplemental material, which is hosted on <a href="https://github.com/amlalejini/evolutionary-consequences-of-plasticity">GitHub</a> <span class="citation" data-cites="supplemental_material">(<strong>supplemental_material?</strong>)</span>. Additionally, our experimental data is available on the Open Science Framework at <a href="https://osf.io/sav2c/">https://osf.io/sav2c/</a> <span class="citation" data-cites="osf_data">(<strong>osf_data?</strong>)</span>.</p>
<h1 id="results">Results</h1>
<h2 id="the-evolution-of-adaptive-phenotypic-plasticity-slows-evolutionary-change-in-fluctuating-environments">The evolution of adaptive phenotypic plasticity slows evolutionary change in fluctuating environments</h2>
<figure>
<embed src="media/evolutionary-change-magnitude-panel.pdf" id="fig:evolutionary-dynamics-magnitude" style="width:100.0%" /><figcaption aria-hidden="true"> <strong>Magnitude of evolutionary change.</strong> Raincloud plots <span class="citation" data-cites="allen_raincloud_2019">(Allen et al. 2019)</span> of (a) coalescence event count, (b) mutation count, and (c) phenotypic volatility, See Table <a href="#tab:metrics-definitions" data-reference-type="ref" data-reference="tab:metrics-definitions">1</a> for descriptions of each metric. Each plot is annotated with statistically significant comparisons (Bonferroni-corrected pairwise Wilcoxon rank-sum tests). Note that adaptive phenotypic plasticity evolved in 42 of 100 replicates from the PLASTIC treatment during phase one of this experiment; we used this more limited group to found the 42 phase-two PLASTIC replicates from which we report these PLASTIC data. </figcaption>
</figure>
<p>In experimental , we tested whether the evolution of adaptive phenotypic plasticity constrained or promoted subsequent evolutionary change in a fluctuating environment. First, we compared the total amount of evolutionary change in populations evolved under the PLASTIC, NON-PLASTIC, and STATIC treatments as measured by coalescence event count, mutation count, and phenotypic volatility (Figure <a href="#fig:evolutionary-dynamics-magnitude" data-reference-type="ref" data-reference="fig:evolutionary-dynamics-magnitude">3</a>). According to each of these metrics, NON-PLASTIC populations experienced a larger magnitude of evolutionary change than either PLASTIC or STATIC populations. We observed significantly higher coalescence event counts in NON-PLASTIC populations than in PLASTIC or STATIC populations (Figure <a href="#fig:evolutionary-dynamics-magnitude" data-reference-type="ref" data-reference="fig:evolutionary-dynamics-magnitude">3</a>). NON-PLASTIC lineages had significantly higher mutation counts (Figure <a href="#fig:evolutionary-dynamics-magnitude" data-reference-type="ref" data-reference="fig:evolutionary-dynamics-magnitude">3</a>b) and phenotypic volatility than PLASTIC or STATIC lineages (Figure <a href="#fig:evolutionary-dynamics-magnitude" data-reference-type="ref" data-reference="fig:evolutionary-dynamics-magnitude">3</a>c).</p>
<p>Changing environments have been shown to increase generational turnover in Avida populations <span class="citation" data-cites="canino-koning_evolution_2016">(Canino-Koning, Wiser, and Ofria 2016)</span>, which could explain why we observe a larger magnitude of evolutionary change at the end of 200,000 updates of evolution in NON-PLASTIC populations. Indeed, we found that significantly more generations of evolution elapsed in NON-PLASTIC populations (mean of <span class="math inline">41090 ± 2702</span> std. dev.) than in PLASTIC (mean of <span class="math inline">31016 ± 2615</span> std. dev.) or STATIC (mean of <span class="math inline">30002 ± 3011</span> std. dev.) populations during phase 2A (corrected Wilcoxon rank-sum tests, p <span class="math inline"> &lt; 10<sup> − 4</sup></span>).</p>
<p>To evaluate whether increased generational turnover explains the greater magnitude of evolutionary change in NON-PLASTIC populations, we examined the average number of generations between coalescence events and the mutational stability of lineages (Table <a href="#tab:metrics-definitions" data-reference-type="ref" data-reference="tab:metrics-definitions">1</a>). A coalescence event indicates a selective sweep, which is a hallmark of adaptive evolutionary change. Mutational stability measures the frequency that mutations cause phenotypic changes along a lineage. We expect that static conditions should favor fit lineages with high mutational stability that no longer undergo rapid adaptive change and hence do not trigger frequent coalescence events. Under fluctuating conditions, however, lineages must be composed of plastic organisms if they are to maintain high fitness and mutational stability. Without plasticity, we expect these conditions to produce lineages with low stability and frequent coalescence events as populations must continually readapt.</p>
<figure>
<embed src="media/evolutionary-change-pace-panel.pdf" id="fig:evolutionary-dynamics-rate" style="width:66.0%" /><figcaption aria-hidden="true"> <strong>Pace of evolutionary change.</strong> Raincloud plots of (a) average number of generations between coalescence events, and (b) mutational stability (Table <a href="#tab:metrics-definitions" data-reference-type="ref" data-reference="tab:metrics-definitions">1</a>). Each plot is annotated with statistically significant comparisons (Bonferroni-corrected pairwise Wilcoxon rank-sum tests). </figcaption>
</figure>
<p>On average, significantly fewer generations elapsed between coalescence events in NON-PLASTIC populations than in either PLASTIC or STATIC populations (Figure <a href="#fig:evolutionary-dynamics-rate" data-reference-type="ref" data-reference="fig:evolutionary-dynamics-rate">4</a>a). We also found that both STATIC and PLASTIC lineages exhibited higher mutational stability relative to that of NON-PLASTIC lineages (Figure <a href="#fig:evolutionary-dynamics-rate" data-reference-type="ref" data-reference="fig:evolutionary-dynamics-rate">4</a>b); that is, mutations more often caused phenotypic changes along NON-PLASTIC lineages. Overall, our results indicate that NON-PLASTIC populations underwent more rapid (and thus a greater amount of) evolutionary changes than either PLASTIC or STATIC populations.</p>
<p>While both STATIC and PLASTIC lineages both exhibited high mutational stability, we found that STATIC lineages exhibited higher mutational stability than PLASTIC lineages (Figure <a href="#fig:evolutionary-dynamics-rate" data-reference-type="ref" data-reference="fig:evolutionary-dynamics-rate">4</a>b). Overall, there were few instances of mutations that caused a change in phenotypic profile across all PLASTIC lineages. However, of these mutations, we found that over 80% (83 out of 102) of changes to phenotypic profiles were cryptic, as they occurred in traits that would not have been expressed when the organism was alive but would have been expressed if the environment changed.</p>
<figure>
<embed src="media/architecture/locus_slice_combined.pdf" id="fig:architecture_locus_functionality" style="width:100.0%" /><figcaption aria-hidden="true"> <strong>todo title.</strong> Representative samples from each condition at the end of phase two. The 100-loci genome of each sample is shown, with colors indicating what functionality a locus has for each of the six base tasks. </figcaption>
</figure>
<figure>
<embed src="media/architecture/avg_time_between_func_changes_weighted_mean.pdf" id="fig:architecture_locus_change_time" style="width:33.0%" /><figcaption aria-hidden="true"> <strong>todo title.</strong> [todo] </figcaption>
</figure>
<p>Next, we performed knockout experiments to investigate how genetic architectures evolved under the PLASTIC, NON-PLASTIC, and STATIC treatment regimes. Figure <a href="#fig:architecture_locus_functionality" data-reference-type="ref" data-reference="fig:architecture_locus_functionality">5</a> shows the functionality of each locus in the most abundant genotype of the final population, using a representative sample from each treatment. PLASTIC and STATIC genomes perform tasks in both environments, while the NON-PLASTIC genome only performs tasks in the B environment. We see substantial vestigial task machinery in environment A for the NON-PLASTIC genome, indicating that the genome previously performed environment A tasks but some other locus changed to “switch off” those tasks while leaving the sites intact. Indeed, Figure <a href="#fig:architecture_locus_change_time" data-reference-type="ref" data-reference="fig:architecture_locus_change_time">6</a> demonstrates that, when looking at locus functionality over an entire lineage, loci change function significantly faster (and thus more often) in the NON-PLASTIC treatment [stats]. This supports our hypothesis that the fluctuating environment forces NON-PLASTIC populations to rely on mutations to continuously readapt, while organisms that are adaptively plastic change their phenotype according to environmental stimuli and thus do not require genetic changes to readapt.</p>
<p>In general, the evolution of adaptive plasticity stabilized PLASTIC treatment populations against environmental fluctuations, and their evolutionary dynamics more closely resembled those of populations evolving in a static environment. We observed no significant difference in the number and frequency of coalescence events in PLASTIC and STATIC populations. We did, however, observe small, but statistically significant, differences in each of elapsed generations, mutation counts, and mutational stability between PLASTIC and STATIC populations (see supplemental material <span class="citation" data-cites="supplemental_material">(<strong>supplemental_material?</strong>)</span>).</p>
<h2 id="adaptively-plastic-populations-retain-more-novel-tasks-than-non-plastic-populations-in-fluctuating-environments">Adaptively plastic populations retain more novel tasks than non-plastic populations in fluctuating environments</h2>
<figure>
<embed src="media/complex-traits-magnitude-panel.pdf" id="fig:complex-traits-magnitude" style="width:90.0%" /><figcaption aria-hidden="true"> <strong>Novel task evolution.</strong> Raincloud plots of (a) final novel task count, (b) novel task discovery, and (c) novel task loss. See Table <a href="#tab:metrics-definitions" data-reference-type="ref" data-reference="tab:metrics-definitions">1</a> for descriptions of each metric. Each plot is annotated with statistically significant comparisons (Bonferroni-corrected pairwise Wilcoxon rank-sum tests). Note that adaptive phenotypic plasticity evolved in 42 of 100 replicates from the PLASTIC treatment during phase one of Experiment II; we used this more limited group to seed the resulting 42 phase-two PLASTIC replicates. </figcaption>
</figure>
<p>In experimental phase 2B, we evaluated how evolution of adaptive phenotypic plasticity influences the ability of populations to evolve and retain novel adaptive traits. We have shown that adaptive plasticity constrains the rate of evolutionary change in fluctuating environments. However, it is unclear how this dynamic influences the evolution of novel tasks. Based on their relative rates of evolutionary change, we might expect NON-PLASTIC-treatment populations to evolve more novel tasks than PLASTIC-treatment populations. But, how much of the evolutionary change in NON-PLASTIC-treatment populations is useful for exploring novel regions of the fitness landscape versus repeatedly re-treading previously explored regions of the fitness landscape?</p>
<p>Figure <a href="#fig:complex-traits-magnitude" data-reference-type="ref" data-reference="fig:complex-traits-magnitude">7</a> shows the final novel task count, novel task discovery, and novel task loss for the PLASTIC, NON-PLASTIC, and STATIC treatments. Final novel task count measures the number of novel tasks performed by a representative organism in the final population of each replicate, which indicates how well the final population exploits the fitness landscape. Novel task discovery is the number of types of novel tasks ever performed by an organism along a representative lineage, which sheds light on how well lineages explore the fitness landscape. Finally, novel task loss is the number of times along a representative lineage that a novel task is performed by a parent but not its offspring, which measures a lineage’s capacity to maintain novel tasks.</p>
<p>We found that both PLASTIC and STATIC populations had significantly higher final task counts than NON-PLASTIC populations at the end of the experiment (Figure <a href="#fig:complex-traits-magnitude" data-reference-type="ref" data-reference="fig:complex-traits-magnitude">7</a>a). This, however, was not due to PLASTIC and STATIC lineages exploring a larger breadth of the fitness landscape, as NON-PLASTIC lineages exhibited significantly higher novel task discovery than either PLASTIC or STATIC lineages (Figure <a href="#fig:complex-traits-magnitude" data-reference-type="ref" data-reference="fig:complex-traits-magnitude">7</a>b). Indeed, we found that NON-PLASTIC lineages repeatedly failed to retain evolved novel tasks, exhibiting significantly higher novel task loss than either PLASTIC or STATIC lineages (Figure <a href="#fig:complex-traits-magnitude" data-reference-type="ref" data-reference="fig:complex-traits-magnitude">7</a>c).</p>
<figure>
<embed src="media/complex-traits-pace-panel.pdf" id="fig:complex-traits-rate" style="width:66.0%" /><figcaption aria-hidden="true"> <strong>Rate of novel task evolution.</strong> Raincloud plots of (a) novel task discovery frequency and (b) novel task loss frequency. Each plot is annotated with statistically significant comparisons (Bonferroni-corrected pairwise Wilcoxon rank-sum tests). </figcaption>
</figure>
<p>As noted previously, a larger number of generations elapsed in NON-PLASTIC populations than in PLASTIC or STATIC populations during our experiment. Are NON-PLASTIC lineages discovering and losing novel tasks more frequently than PLASTIC or STATIC lineages, or are our observations a result of differences in generational turnover? To tease this apart, we examined the frequencies of novel task discovery and novel task loss along representative lineages by dividing novel task discovery and novel task loss measurements by the number of elapsed generations. We found no significant difference in the frequency of novel task discovery between NON-PLASTIC and STATIC lineages, and we found that PLASTIC lineages had a lower frequency of novel task discovery than STATIC lineages (Figure <a href="#fig:complex-traits-rate" data-reference-type="ref" data-reference="fig:complex-traits-rate">8</a>a). Therefore, we cannot reject the possibility that the larger magnitude of task discovery in NON-PLASTIC lineages was driven by a larger number of elapsed generations. NON-PLASTIC lineages had a higher frequency of task loss than either PLASTIC or STATIC lineages, and PLASTIC lineages tended to have a lower frequency of novel task loss than STATIC lineages (Figure <a href="#fig:complex-traits-rate" data-reference-type="ref" data-reference="fig:complex-traits-rate">8</a>b).</p>
<p>Next, we examined the frequency at which novel task loss along lineages co-occurred with a change in base task performance (<em>i.e.</em>, the loss or gain of any of the six base tasks). Across all NON-PLASTIC dominant lineages, over 97% (10998 out of 11229) of instances of novel task loss co-occurred with a simultaneous change in base task profile. In contrast, across all PLASTIC and STATIC dominant lineages, we observed that approximately 20% (29 out of 142) and 2% (13 out of 631), respectively, of instances of novel task loss co-occurred with a simultaneous change in base task profile.</p>
<h2 id="non-plastic-lineages-accumulate-more-deleterious-instructions-in-fluctuating-environments">Non-plastic lineages accumulate more deleterious instructions in fluctuating environments</h2>
<p>Phenotypic plasticity allows for possibility of accumulating of genetic variation in genomic regions that are unexpressed under contemporary environmental conditions, which could lead to the fixation of deleterious instructions in PLASTIC-treatment populations. However, our results thus far indicate that PLASTIC-treatment populations accumulate <em>fewer</em> deleterious mutations than NON-PLASTIC treatment populations in fluctuating environments. That is, NON-PLASTIC lineages accumulated more total genetic changes (Figure <a href="#fig:evolutionary-dynamics-magnitude" data-reference-type="ref" data-reference="fig:evolutionary-dynamics-magnitude">3</a>), and the high rates of novel task loss in NON-PLASTIC lineages (Figure <a href="#fig:complex-traits-rate" data-reference-type="ref" data-reference="fig:complex-traits-rate">8</a>) may also indicate their increased susceptibility to accumulating deleterious mutations relative to PLASTIC lineages.</p>
<p>In experimental phase 2C, we tested whether the evolution of adaptive phenotypic plasticity increases deleterious instruction accumulation in genomes by measuring poisonous task performance in representative organisms and along representative lineages. Organisms can perform the poisonous task by executing a <code>poison</code> instruction, which reduces their fitness. At the beginning of phase two, the <code>poison</code> instruction is not present in the population as it was not part of the instruction set during the first phase of evolution. If a <code>poison</code> instruction fixes in a population, it must be the result of evolutionary dynamics during phase 2C, as no founding organism’s genome contained the <code>poison</code> instruction. Selection should purge mutations that increase <code>poison</code> instruction execution in the offspring phenotype, unless those mutations accumulate as cryptic variation in plastic genomes or hitchhike to fixation with linked beneficial mutations.</p>
<figure>
<embed src="media/poison-accumulation-panel.pdf" id="fig:deleterious-hitchhiking" style="width:100.0%" /><figcaption aria-hidden="true"> <strong>Deleterious instruction accumulation.</strong> Raincloud plots of (a) poisonous task acquisition, (b) poisonous task acquisition frequency, and (c) the proportion of mutations that increase poisonous task performance along a lineage that co-occur with another change in phenotypic profile. Each plot is annotated with statistically significant comparisons (Bonferroni-corrected pairwise Wilcoxon rank-sum tests). Note that adaptive phenotypic plasticity evolved in 43 of 100 replicates from the PLASTIC treatment during phase one of Experiment III; we used this more limited group to seed the 43 phase-two PLASTIC replicates. </figcaption>
</figure>
<p>At the end of our experiment, no representative organisms from the PLASTIC or STATIC treatments performed the poisonous task under any environmental condition; however, representative organisms from 14 replicates of the NON-PLASTIC treatment performed the poisonous task at least once. Figure <a href="#fig:deleterious-hitchhiking" data-reference-type="ref" data-reference="fig:deleterious-hitchhiking">9</a> shows the poisonous task acquisition rate for each treatment, which measures the number of instances along a lineage where a mutation causes an offspring to perform the poisonous task more often than its parent. We found that NON-PLASTIC lineages contained significantly more offspring that performed the poisonous task more often than their parent as compared to offspring along PLASTIC or STATIC lineages (<a href="#fig:deleterious-hitchhiking" data-reference-type="ref" data-reference="fig:deleterious-hitchhiking">9</a>a). This result does not change when we normalize each poisonous task acquisition value by the number of generations represented in the given lineage (Figure <a href="#fig:deleterious-hitchhiking" data-reference-type="ref" data-reference="fig:deleterious-hitchhiking">9</a>b).</p>
<p>Next, we examined the frequency that mutations that increase <code>poison</code> execution co-occurred with mutations that change the base task profile of offspring along representative lineages. Across all NON-PLASTIC representative lineages, we found that approximately 49% (956 out of 1916) mutations that increased <code>poison</code> instruction execution co-occurred with mutations that changed the base task profile. Along all representative lineages from the PLASTIC treatment, only 18 offspring sustained a mutation that increased task poisonous task performance relative to their parent, and none of these mutations simultaneously co-occurred with a change in base task profile. Likewise, along all representative lineages from the STATIC treatment, only 58 offspring sustained a mutation that increased poisonous task performance, and none of these mutations simultaneously co-occurred with a change in base task profile. We did not find compelling evidence that the few mutations that increased <code>poison</code> task performance occurred as cryptic variation in PLASTIC lineages.</p>
<p>We repeated this experiment with 3% and 30% metabolic rate penalties associated with the poisonous task, which produced consistent experimental results <span class="citation" data-cites="supplemental_material">(<strong>supplemental_material?</strong>)</span>.</p>
<h1 id="discussion">Discussion</h1>
<p>In this work, we used evolving populations of digital organisms to determine how adaptive phenotypic plasticity alters subsequent evolutionary dynamics and influences evolutionary outcomes in fluctuating environments. First, we examined the evolutionary histories of plastic and non-plastic populations to test whether the evolution of adaptive plasticity promotes or constrains subsequent evolutionary change. Next, we evaluated how adaptive plasticity influences fitness landscape exploration and exploitation by testing whether plastic populations are better able to evolve and then maintain novel traits. Finally, we tested if the evolution of adaptive plasticity increases the potential for deleterious instructions to accumulate in evolving genomes.</p>
<p>Overall, our results indicate that adaptive plasticity can improve evolution’s ability to maintain and refine novel traits, though with the tradeoff of reducing evolutionary exploration of the fitness landscape. Additionally, we found no evidence that adaptive plasticity increased the potential for deleterious instructions to accumulate in genomes. Instead, the genomes of non-plastic organisms that evolved in an identical fluctuating environment accumulated more deleterious instructions than that of the genomes of adaptively plastic organisms. These dynamics appear to be driven by the stabilizing effect that adaptive plasticity had on population dynamics rather than plasticity’s effect on genetic architecture or regulation.</p>
<h2 id="the-rate-of-evolutionary-change">The rate of evolutionary change</h2>
<p>Adaptively plastic populations experienced fewer selective sweeps and fewer total genetic changes relative to non-plastic populations evolving under the same environmental conditions (Figure <a href="#fig:evolutionary-dynamics-magnitude" data-reference-type="ref" data-reference="fig:evolutionary-dynamics-magnitude">3</a>). Plastic populations adapted to the fluctuating environmental regime by evolving to sense environmental changes and regulate their metabolism (task performance) in response to such changes, which also stabilized these populations against fluctuations. Indeed, across all three of our experiments, the evolutionary dynamics of plastic populations were more similar to that of populations evolving in a static environment than to that of non-plastic populations evolving in an identical fluctuating environment.</p>
<p>Adaptive phenotypes in ENV-A were maladaptive in ENV-B and vice versa. As such, non-plastic generalists that performed all tasks or performed no tasks at all did not evolve in any of the fluctuating environmental regimes. Selection against non-plastic generalists may be attributed to competition with phenotypic specialists that have a much larger fitness advantage for performing environment-specific tasks. In non-plastic populations where plasticity was disallowed, we hypothesize that strong selection on task specialization after each environmental change drove the repeated fixation of beneficial mutations (that alter an organism’s phenotypic profile). This hypothesis is supported by the increased frequency of coalescence events in these populations (Figure <a href="#fig:evolutionary-dynamics-rate" data-reference-type="ref" data-reference="fig:evolutionary-dynamics-rate">4</a>) as well as increased rates of genetic and phenotypic changes observed along the lineages of non-plastic organisms.</p>
<p>Analysis of the evolved genetic architectures further supports our hypothesis that the non-plastic populations relied on mutations to continuously readapt to the fluctuating environment. This aligns with previous work, which has shown that, in the absence of plasticity, fluctuating environments steer populations toward genotypes that readily mutate to alternative phenotypes <span class="citation" data-cites="lalejini_evolutionary_2016 canino-koning_evolution_2016">(Lalejini and Ofria 2016; Canino-Koning, Wiser, and Ofria 2016)</span>. Indeed, <span class="citation" data-cites="canino-koning_evolution_2016">(Canino-Koning, Wiser, and Ofria 2016)</span> also observed that genomes evolved in harsh cyclic environments often contained vestigial fragments of genetic material adapted to prior environments, which we see in the non-plastic populations. [blah...]</p>
<p>This study is the first in-depth empirical investigation into how the <em>de novo</em> evolution of adaptive plasticity shifts the course of subsequent evolution in a cyclic environment. The evolutionary dynamics that we observed in non-plastic populations, however, are consistent with results from previous digital evolution studies. Consistent with our findings, <span class="citation" data-cites="dolson_interpreting_2020">(Dolson et al. 2020)</span> showed that non-plastic populations that were evolved in cyclically changing environments exhibited higher phenotypic volatility and accumulated more mutations than that of populations evolved in static conditions.</p>
<p>Our results are also consistent with conventional evolutionary theory. A trait’s evolutionary response to selection depends on the strength of directional selection and on the amount of genetic variation for selection to act upon <span class="citation" data-cites="lande_measurement_1983 zimmer_evolution_2013">(Lande and Arnold 1983; Zimmer and Emlen 2013)</span>. In our experiments, non-plastic populations repeatedly experienced strong directional selection to toggle which tasks were expressed after each environmental change. As such, retrospective analyses of successful lineages revealed rapid evolutionary responses (that is, high rates of genetic and phenotypic changes). Evolved adaptive plasticity shielded populations from strong directional selection when the environment changed by eliminating the need for a rapid evolutionary response to toggle task expression. Indeed, both theoretical and empirical studies have shown that adaptive plasticity can constrain evolutionary change by weakening directional selection on evolving populations <span class="citation" data-cites="price_role_2003 paenke_influence_2007 ghalambor_non-adaptive_2015">(Price, Qvarnström, and Irwin 2003; Paenke, Sendhoff, and Kawecki 2007; Cameron K. Ghalambor et al. 2015)</span>.</p>
<h2 id="the-evolution-and-maintenance-of-novel-tasks">The evolution and maintenance of novel tasks</h2>
<p>In fluctuating environments, non-plastic populations explored a larger area of the fitness landscape than adaptively plastic populations, as measured by the number of novel task discovery (Figure <a href="#fig:complex-traits-magnitude" data-reference-type="ref" data-reference="fig:complex-traits-magnitude">7</a>b). Despite lower overall novel task discovery in adaptively plastic populations, they better exploited the fitness landscape, retaining a greater number of novel tasks than non-plastic populations evolving under identical environmental conditions (Figure <a href="#fig:complex-traits-magnitude" data-reference-type="ref" data-reference="fig:complex-traits-magnitude">7</a>a). Evolution in non-plastic populations was dominated by numerous bouts of strong directional selection, driven by repeated environmental change. After each change, the performance of the six base tasks needed to be realigned to the environment. In our experiment, novel tasks were less important to survival than the fluctuating base tasks. In non-plastic populations, mutations that improve an offspring’s fitness after an environmental change are extremely beneficial, and as such beneficial mutations fix, they can carry with them co-occurring deleterious mutations that knock out novel tasks. Indeed, we found that mutations associated with novel task loss along representative lineages from non-plastic populations co-occurred with mutations that helped offspring adapt to environmental changes 97% of the time.</p>
<p>Temporary environmental changes can improve fitness landscape exploration and exploitation in evolving populations of non-plastic digital organisms <span class="citation" data-cites="nahum_improved_2017">(Nahum et al. 2017)</span>. In our system, however, we found that <em>repeated</em> fluctuations reduced the ability of non-plastic populations to maintain and exploit tasks; that said, we did find that repeated fluctuations may improve overall task discovery by increasing generational turnover. Consistent with our findings, <span class="citation" data-cites="canino-koning_fluctuating_2019">(Canino-Koning, Wiser, and Ofria 2019)</span> found that non-plastic populations of digital organisms evolving in a harsh cyclic environment maintained fewer novel traits than populations evolving in static environments.</p>
<p>Our results suggest that adaptive phenotypic plasticity can improve the potential for populations to exploit novel resources by stabilizing them against stressful environmental changes. The stability that we observe may also lend some support to the hypothesis that phenotypic plasticity can rescue populations from extinction under changing environmental conditions <span class="citation" data-cites="chevin_adaptation_2010">(Chevin, Lande, and Mace 2010)</span>.</p>
<p>Our data do not necessarily provide evidence for or against the genes as followers hypothesis. The genes as followers hypothesis focuses on contexts where plastic populations experience novel or abnormally stressful environmental change. However, in our system, environmental changes were cyclic (not novel), and the magnitude of changes were consistent for the entirety of the experiment (so none were abnormally stressful). Further, the introduction of novel tasks during the second phase of the experiment merely added additional static opportunities for fitness improvement and did not change the meaning of existing environmental cues.</p>
<h2 id="the-accumulation-of-deleterious-instructions">The accumulation of deleterious instructions</h2>
<p>We found that non-plastic lineages that evolved in a fluctuating environment exhibited both larger totals and higher rates of deleterious instruction (<code>poison</code>) accumulation than that of adaptively plastic lineages (Figure <a href="#fig:deleterious-hitchhiking" data-reference-type="ref" data-reference="fig:deleterious-hitchhiking">9</a>). We did not find evidence of <code>poison</code> instructions accumulating as cryptic variation in adaptively plastic lineages. We hypothesize that deleterious genetic hitchhiking drove <code>poison</code> instruction accumulation along non-plastic lineages in changing environments. In asexual populations without horizontal gene transfer, all co-occurring mutations are linked. As such, deleterious mutations linked with a stronger beneficial mutation (<em>i.e.</em>, a driver) can sometimes “hitchhike” to fixation <span class="citation" data-cites="smith_hitch-hiking_1974 van_den_bergh_experimental_2018 buskirk_hitchhiking_2017">(Smith and Haigh 1974; Van den Bergh et al. 2018; Buskirk, Peace, and Lang 2017)</span>. Natural selection normally prevents deleterious mutations from reaching high frequencies, as such mutants would be outcompeted. However, when a beneficial mutation sweeps to fixation in a clonal population, it carries along any linked genetic material, including other beneficial, neutral, or deleterious mutations <span class="citation" data-cites="barton_genetic_2000 smith_hitch-hiking_1974">(<strong>barton_genetic_2000?</strong>; Smith and Haigh 1974)</span>.</p>
<p>Across our experiments, the frequency of selective sweeps in non-plastic populations provided additional opportunities for genetic hitchhiking with each environmental change. Indeed, representative lineages from non-plastic populations in the cyclic environment exhibited higher mutation accumulation (Figure <a href="#fig:evolutionary-dynamics-magnitude" data-reference-type="ref" data-reference="fig:evolutionary-dynamics-magnitude">3</a>b), novel trait loss (Figure <a href="#fig:complex-traits-magnitude" data-reference-type="ref" data-reference="fig:complex-traits-magnitude">7</a>c), and deleterious instruction accumulation (Figure <a href="#fig:deleterious-hitchhiking" data-reference-type="ref" data-reference="fig:deleterious-hitchhiking">9</a>) than their plastic counterparts. In aggregate, we found that many (<span class="math inline">∼</span>49%; 956 / 1916) mutations that increased <code>poison</code> instruction execution in offspring co-occurred with mutations that provided an even stronger benefit by adapting the offspring to an environmental change. This rate of co-occurrence is conservative because we did not analyze mutations that became linked in different generations.</p>
<p>We found that adaptive phenotypic plasticity reduced <code>poison</code> instruction accumulation by reducing the rate of evolutionary change, which in turn reduced opportunities for <code>poison</code> instructions to hitchhike to fixation. We did not find compelling evidence of cryptic variation harboring <code>poison</code> instructions in adaptively plastic lineages. We have two hypotheses for why we did not observe the accumulation of <code>poison</code> instructions in unexpressed plastic responses. First, the period of time between environmental changes was too fast for variants carrying unexpressed <code>poison</code> instructions to reach high frequencies before the environment changed, after which such variants would have been outcompeted. Indeed, we tuned the frequency of environmental fluctuations so that genes that needed to function appropriately in the off environment were able to remain in the population despite relaxed selection. Second, the genetic mechanisms of plasticity that evolve in Avida are typically well-integrated and highly specific; that is, plastic genomes usually adjust their phenotypic expression by toggling a minimal number of key instructions. As such, there is little genomic space for variation to accumulate in preexisting (but unexpressed) regulated regions.</p>
<h2 id="limitations-and-future-directions">Limitations and future directions</h2>
<p>Our conclusions are limited to <em>adaptively</em> plastic populations. We did not explore the effects of non-adaptive plasticity where environmental changes induced phenotypes that were further away from the local optimum. Non-adaptive plasticity can increase a population’s extinction risk, especially if the misaligned plastic response is strongly tied to survival or the population is not sufficiently large <span class="citation" data-cites="gomulkiewicz_when_1995 chevin_adaptation_2010">(Gomulkiewicz and Holt 1995; Chevin, Lande, and Mace 2010)</span>. If the population persists, however, non-adaptive plasticity has been shown to be capable of accelerating evolutionary change by increasing the strength of directional selection. <span class="citation" data-cites="ghalambor_non-adaptive_2015">(Cameron K. Ghalambor et al. 2015)</span>.</p>
<p>Environmental cues in our experiments were reliable, and environmental changes were consistent over time; that is, sensory instructions perfectly differentiated between ENV-A and ENV-B, and environmental fluctuations never exposed populations to entirely new conditions. Both the reliability of cues and the timescales of environment switching are known to influence evolutionary outcomes <span class="citation" data-cites="li_digital_2004 boyer_adaptation_2021">(Li and Wilke 2004; Boyer, Hérissant, and Sherlock 2021)</span>. For example, <span class="citation" data-cites="boyer_adaptation_2021">(Boyer, Hérissant, and Sherlock 2021)</span> evolved populations of <em>Saccharomyces cerevisiae</em> in an environment that fluctuated between two growth conditions, observing that both environmental predictability and switching rate influenced the rates of evolutionary responses as well as adaptive outcomes at the genotypic and phenotypic levels. In adaptively plastic populations, environmental switching rate can influence how plastic responses are maintained, including their genetic architecture as well as their likelihood of maintenance. Here, we lay the groundwork for using digital evolution experiments to investigate the evolutionary consequences of phenotypic plasticity in a range of contexts, including different forms of plasticity (<em>e.g.</em>, adaptive versus non-adaptive), more complex environments with more than two possible states, stochastic environmental changes, and different environment switching rates.</p>
<p>We focused our analyses on the lineages of organisms with the most abundant genotype in the final population. These successful lineages represented the majority of the evolutionary histories of populations at the end of our experiment, as populations did not exhibit long-term coexistence of different clades. Our analyses, therefore, gave us an accurate picture of what fixed in the population. We did not, however, examine the lineages of extinct clades. Future work will extend our analyses to include extinct lineages, giving us a more complete view of evolutionary history, which may allow us to better distinguish adaptively plastic populations from populations evolving in a static environment.</p>
<p>As with any wet-lab experiment, our results are in the context of a particular model organism: “Avidian” self-replicating computer programs. Digital organisms in Avida regulate responses to environmental cues using a combination of sensory instructions and conditional logic instructions (<code>if</code> statements). The <code>if</code> instructions conditionally execute a single instruction depending on previous computations and the state of memory. As such, plastic genomes typically regulate a small number of key instructions that, when executed, change the expressed phenotype as opposed to large, sequences of co-regulated instructions. This bias may limit the accumulation of hidden genetic variation in Avida genomes. However, as there are many model biological organisms, there are many model digital organisms that have different regulatory mechanisms that should be used to test the generality of our results.</p>
<h1 class="unnumbered" id="supplemental-data">Supplemental Data</h1>
<h1 class="unnumbered" id="data-availability-statement">Data Availability Statement</h1>
<h1 class="unnumbered" id="author-contributions">Author Contributions</h1>
<h1 class="unnumbered" id="funding">Funding</h1>
<h1 class="unnumbered" id="conflict-of-interest-statement">Conflict of Interest Statement</h1>
<p>The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.</p>
<h1 class="unnumbered" id="acknowledgments">Acknowledgments</h1>
<div id="refs" class="references csl-bib-body hanging-indent" role="doc-bibliography">
<div id="ref-allen_raincloud_2019" class="csl-entry" role="doc-biblioentry">
Allen, Micah, Davide Poggiali, Kirstie Whitaker, Tom Rhys Marshall, and Rogier A. Kievit. 2019. <span>“Raincloud Plots: A Multi-Platform Tool for Robust Data Visualization.”</span> <em>Wellcome Open Research</em> 4 (April): 63. <a href="https://doi.org/10.12688/wellcomeopenres.15191.1">https://doi.org/10.12688/wellcomeopenres.15191.1</a>.
</div>
<div id="ref-ancel_undermining_2000" class="csl-entry" role="doc-biblioentry">
Ancel, Lauren W. 2000. <span>“Undermining the <span>Baldwin</span> <span>Expediting</span> <span>Effect</span>: <span>Does</span> <span>Phenotypic</span> <span>Plasticity</span> <span>Accelerate</span> <span>Evolution</span>?”</span> <em>Theoretical Population Biology</em> 58 (4): 307–19. <a href="https://doi.org/10.1006/tpbi.2000.1484">https://doi.org/10.1006/tpbi.2000.1484</a>.
</div>
<div id="ref-barrett_adaptation_2008" class="csl-entry" role="doc-biblioentry">
Barrett, R, and D Schluter. 2008. <span>“Adaptation from Standing Genetic Variation.”</span> <em>Trends in Ecology &amp; Evolution</em> 23 (1): 38–44. <a href="https://doi.org/10.1016/j.tree.2007.09.008">https://doi.org/10.1016/j.tree.2007.09.008</a>.
</div>
<div id="ref-beaumont_experimental_2009" class="csl-entry" role="doc-biblioentry">
Beaumont, Hubertus J. E., Jenna Gallie, Christian Kost, Gayle C. Ferguson, and Paul B. Rainey. 2009. <span>“Experimental Evolution of Bet Hedging.”</span> <em>Nature</em> 462 (7269): 90–93. <a href="https://doi.org/10.1038/nature08504">https://doi.org/10.1038/nature08504</a>.
</div>
<div id="ref-boyer_adaptation_2021" class="csl-entry" role="doc-biblioentry">
Boyer, Sébastien, Lucas Hérissant, and Gavin Sherlock. 2021. <span>“Adaptation Is Influenced by the Complexity of Environmental Change During Evolution in a Dynamic Environment.”</span> Edited by Geraldine Butler. <em>PLOS Genetics</em> 17 (1): e1009314. <a href="https://doi.org/10.1371/journal.pgen.1009314">https://doi.org/10.1371/journal.pgen.1009314</a>.
</div>
<div id="ref-buskirk_hitchhiking_2017" class="csl-entry" role="doc-biblioentry">
Buskirk, Sean W., Ryan Emily Peace, and Gregory I. Lang. 2017. <span>“Hitchhiking and Epistasis Give Rise to Cohort Dynamics in Adapting Populations.”</span> <em>Proceedings of the National Academy of Sciences</em> 114 (31): 8330–35. <a href="https://doi.org/10.1073/pnas.1702314114">https://doi.org/10.1073/pnas.1702314114</a>.
</div>
<div id="ref-canino-koning_evolution_2016" class="csl-entry" role="doc-biblioentry">
Canino-Koning, Rosangela, Michael J. Wiser, and Charles Ofria. 2016. <span>“The <span>Evolution</span> of <span>Evolvability</span>: <span>Changing</span> <span>Environments</span> <span>Promote</span> <span>Rapid</span> <span>Adaptation</span> in <span>Digital</span> <span>Organisms</span>.”</span> In <em>Proceedings of the <span>Artificial</span> <span>Life</span> <span>Conference</span> 2016</em>, 268–75. Cancun, Mexico: MIT Press. <a href="https://doi.org/10.7551/978-0-262-33936-0-ch047">https://doi.org/10.7551/978-0-262-33936-0-ch047</a>.
</div>
<div id="ref-canino-koning_fluctuating_2019" class="csl-entry" role="doc-biblioentry">
———. 2019. <span>“Fluctuating Environments Select for Short-Term Phenotypic Variation Leading to Long-Term Exploration.”</span> Edited by Laurent Keller. <em>PLOS Computational Biology</em> 15 (4): e1006445. <a href="https://doi.org/10.1371/journal.pcbi.1006445">https://doi.org/10.1371/journal.pcbi.1006445</a>.
</div>
<div id="ref-chadwick_parasite-mediated_2005" class="csl-entry" role="doc-biblioentry">
Chadwick, William, and Tom J Little. 2005. <span>“A Parasite-Mediated Life-History Shift in <em><span>Daphnia</span> Magna</em>.”</span> <em>Proceedings of the Royal Society B: Biological Sciences</em> 272 (1562): 505–9. <a href="https://doi.org/10.1098/rspb.2004.2959">https://doi.org/10.1098/rspb.2004.2959</a>.
</div>
<div id="ref-chevin_when_2010" class="csl-entry" role="doc-biblioentry">
Chevin, Luis-Miguel, and Russell Lande. 2010. <span>“When Do Adaptive Plasticity and Genetic Evolution Prevent Extinction of a Density‐regulated Population?”</span> <em>Evolution</em> 64 (4): 1143–50. <a href="https://doi.org/10.1111/j.1558-5646.2009.00875.x">https://doi.org/10.1111/j.1558-5646.2009.00875.x</a>.
</div>
<div id="ref-chevin_adaptation_2010" class="csl-entry" role="doc-biblioentry">
Chevin, Luis-Miguel, Russell Lande, and Georgina M. Mace. 2010. <span>“Adaptation, <span>Plasticity</span>, and <span>Extinction</span> in a <span>Changing</span> <span>Environment</span>: <span>Towards</span> a <span>Predictive</span> <span>Theory</span>.”</span> Edited by Joel G. Kingsolver. <em>PLoS Biology</em> 8 (4): e1000357. <a href="https://doi.org/10.1371/journal.pbio.1000357">https://doi.org/10.1371/journal.pbio.1000357</a>.
</div>
<div id="ref-clune_investigating_2007" class="csl-entry" role="doc-biblioentry">
Clune, Jeff, Charles Ofria, and Robert T. Pennock. 2007. <span>“Investigating the <span>Emergence</span> of <span>Phenotypic</span> <span>Plasticity</span> in <span>Evolving</span> <span>Digital</span> <span>Organisms</span>.”</span> In <em>Advances in <span>Artificial</span> <span>Life</span></em>, edited by Fernando Almeida e Costa, Luis Mateus Rocha, Ernesto Costa, Inman Harvey, and António Coutinho, 4648:74–83. Berlin, Heidelberg: Springer Berlin Heidelberg. <a href="https://doi.org/10.1007/978-3-540-74913-4_8">https://doi.org/10.1007/978-3-540-74913-4_8</a>.
</div>
<div id="ref-covert_experiments_2013" class="csl-entry" role="doc-biblioentry">
Covert, Arthur W., Richard E. Lenski, Claus O. Wilke, and Charles Ofria. 2013. <span>“Experiments on the Role of Deleterious Mutations as Stepping Stones in Adaptive Evolution.”</span> <em>Proceedings of the National Academy of Sciences</em> 110 (34): E3171–78. <a href="https://doi.org/10.1073/pnas.1313424110">https://doi.org/10.1073/pnas.1313424110</a>.
</div>
<div id="ref-crispo_baldwin_2007" class="csl-entry" role="doc-biblioentry">
Crispo, Erika. 2007. <span>“The <span>Baldwin</span> Effect and Genetic Assimilation: Revisiting Two Mechanisms of Evolutionary Change Mediated by Phenotypic Plasticity.”</span> <em>Evolution</em> 61 (11): 2469–79. <a href="https://doi.org/10.1111/j.1558-5646.2007.00203.x">https://doi.org/10.1111/j.1558-5646.2007.00203.x</a>.
</div>
<div id="ref-dolson_interpreting_2020" class="csl-entry" role="doc-biblioentry">
Dolson, Emily, Alexander Lalejini, Steven Jorgensen, and Charles Ofria. 2020. <span>“Interpreting the <span>Tape</span> of <span>Life</span>: <span>Ancestry</span>-<span>Based</span> <span>Analyses</span> <span>Provide</span> <span>Insights</span> and <span>Intuition</span> about <span>Evolutionary</span> <span>Dynamics</span>.”</span> <em>Artificial Life</em> 26 (1): 58–79. <a href="https://doi.org/10.1162/artl_a_00313">https://doi.org/10.1162/artl_a_00313</a>.
</div>
<div id="ref-dunn_two_2005" class="csl-entry" role="doc-biblioentry">
Dunn, Alison M., John C. Hogg, Andrew Kelly, and Melanie J. Hatcher. 2005. <span>“Two Cues for Sex Determination in <em><span>Gammarus</span> Duebeni</em> : <span>Adaptive</span> Variation in Environmental Sex Determination?”</span> <em>Limnology and Oceanography</em> 50 (1): 346–53. <a href="https://doi.org/10.4319/lo.2005.50.1.0346">https://doi.org/10.4319/lo.2005.50.1.0346</a>.
</div>
<div id="ref-forsman_rethinking_2015" class="csl-entry" role="doc-biblioentry">
Forsman, A. 2015. <span>“Rethinking Phenotypic Plasticity and Its Consequences for Individuals, Populations and Species.”</span> <em>Heredity</em> 115 (4): 276–84. <a href="https://doi.org/10.1038/hdy.2014.92">https://doi.org/10.1038/hdy.2014.92</a>.
</div>
<div id="ref-ghalambor_adaptive_2007" class="csl-entry" role="doc-biblioentry">
Ghalambor, C. K., J. K. McKAY, S. P. Carroll, and D. N. Reznick. 2007. <span>“Adaptive Versus Non-Adaptive Phenotypic Plasticity and the Potential for Contemporary Adaptation in New Environments.”</span> <em>Functional Ecology</em> 21 (3): 394–407. <a href="https://doi.org/10.1111/j.1365-2435.2007.01283.x">https://doi.org/10.1111/j.1365-2435.2007.01283.x</a>.
</div>
<div id="ref-ghalambor_behavior_2010" class="csl-entry" role="doc-biblioentry">
Ghalambor, Cameron K., Lisa M. Angeloni, and Scott P. Carroll. 2010. <span>“Behavior as Phenotypic Plasticity.”</span> In <em>Evolutionary Behavioral Ecology</em>, edited by David Westneat and Charles W. Fox, 90–107. New York, NY: Oxford University Press.
</div>
<div id="ref-ghalambor_non-adaptive_2015" class="csl-entry" role="doc-biblioentry">
Ghalambor, Cameron K., Kim L. Hoke, Emily W. Ruell, Eva K. Fischer, David N. Reznick, and Kimberly A. Hughes. 2015. <span>“Non-Adaptive Plasticity Potentiates Rapid Adaptive Evolution of Gene Expression in Nature.”</span> <em>Nature</em> 525 (7569): 372–75. <a href="https://doi.org/10.1038/nature15256">https://doi.org/10.1038/nature15256</a>.
</div>
<div id="ref-gibert_phenotypic_2019" class="csl-entry" role="doc-biblioentry">
Gibert, Patricia, Vincent Debat, and Cameron K Ghalambor. 2019. <span>“Phenotypic Plasticity, Global Change, and the Speed of Adaptive Evolution.”</span> <em>Current Opinion in Insect Science</em> 35 (October): 34–40. <a href="https://doi.org/10.1016/j.cois.2019.06.007">https://doi.org/10.1016/j.cois.2019.06.007</a>.
</div>
<div id="ref-gibson_uncovering_2004" class="csl-entry" role="doc-biblioentry">
Gibson, Greg, and Ian Dworkin. 2004. <span>“Uncovering Cryptic Genetic Variation.”</span> <em>Nature Reviews Genetics</em> 5 (9): 681–90. <a href="https://doi.org/10.1038/nrg1426">https://doi.org/10.1038/nrg1426</a>.
</div>
<div id="ref-goldsby_evolutionary_2014" class="csl-entry" role="doc-biblioentry">
Goldsby, Heather J., David B. Knoester, Charles Ofria, and Benjamin Kerr. 2014. <span>“The <span>Evolutionary</span> <span>Origin</span> of <span>Somatic</span> <span>Cells</span> Under the <span>Dirty</span> <span>Work</span> <span>Hypothesis</span>.”</span> Edited by Laurent Keller. <em>PLoS Biology</em> 12 (5): e1001858. <a href="https://doi.org/10.1371/journal.pbio.1001858">https://doi.org/10.1371/journal.pbio.1001858</a>.
</div>
<div id="ref-gomulkiewicz_when_1995" class="csl-entry" role="doc-biblioentry">
Gomulkiewicz, Richard, and Robert D. Holt. 1995. <span>“When Does <span>Evolution</span> by <span>Natural</span> <span>Selection</span> <span>Prevent</span> <span>Extinction</span>?”</span> <em>Evolution</em> 49 (1): 201. <a href="https://doi.org/10.2307/2410305">https://doi.org/10.2307/2410305</a>.
</div>
<div id="ref-gupta_study_1982" class="csl-entry" role="doc-biblioentry">
Gupta, Anand P., and R. C. Lewontin. 1982. <span>“A <span>Study</span> of <span>Reaction</span> <span>Norms</span> in <span>Natural</span> <span>Populations</span> of <span>Drosophila</span> Pseudoobscura.”</span> <em>Evolution</em> 36 (5): 934. <a href="https://doi.org/10.2307/2408073">https://doi.org/10.2307/2408073</a>.
</div>
<div id="ref-hendry_key_2016" class="csl-entry" role="doc-biblioentry">
Hendry, Andrew P. 2016. <span>“Key <span>Questions</span> on the <span>Role</span> of <span>Phenotypic</span> <span>Plasticity</span> in <span>Eco</span>-<span>Evolutionary</span> <span>Dynamics</span>.”</span> <em>Journal of Heredity</em> 107 (1): 25–41. <a href="https://doi.org/10.1093/jhered/esv060">https://doi.org/10.1093/jhered/esv060</a>.
</div>
<div id="ref-huey_behavioral_2003" class="csl-entry" role="doc-biblioentry">
Huey, Raymond B., Paul E. Hertz, and B. Sinervo. 2003. <span>“Behavioral <span>Drive</span> Versus <span>Behavioral</span> <span>Inertia</span> in <span>Evolution</span>: <span>A</span> <span>Null</span> <span>Model</span> <span>Approach</span>.”</span> <em>The American Naturalist</em> 161 (3): 357–66. <a href="https://doi.org/10.1086/346135">https://doi.org/10.1086/346135</a>.
</div>
<div id="ref-kruskal_use_1952" class="csl-entry" role="doc-biblioentry">
Kruskal, William H., and W. Allen Wallis. 1952. <span>“Use of <span>Ranks</span> in <span>One</span>-<span>Criterion</span> <span>Variance</span> <span>Analysis</span>.”</span> <em>Journal of the American Statistical Association</em> 47 (260): 583–621. <a href="https://doi.org/10.1080/01621459.1952.10483441">https://doi.org/10.1080/01621459.1952.10483441</a>.
</div>
<div id="ref-lalejini_evolutionary_2016" class="csl-entry" role="doc-biblioentry">
Lalejini, Alexander, and Charles Ofria. 2016. <span>“The <span>Evolutionary</span> <span>Origins</span> of <span>Phenotypic</span> <span>Plasticity</span>.”</span> In <em>Proceedings of the <span>Artificial</span> <span>Life</span> <span>Conference</span> 2016</em>, 372–79. Cancun, Mexico: MIT Press. <a href="https://doi.org/10.7551/978-0-262-33936-0-ch063">https://doi.org/10.7551/978-0-262-33936-0-ch063</a>.
</div>
<div id="ref-lande_measurement_1983" class="csl-entry" role="doc-biblioentry">
Lande, Russell, and Stevan J. Arnold. 1983. <span>“The <span>Measurement</span> of <span>Selection</span> on <span>Correlated</span> <span>Characters</span>.”</span> <em>Evolution</em> 37 (6): 1210. <a href="https://doi.org/10.2307/2408842">https://doi.org/10.2307/2408842</a>.
</div>
<div id="ref-lenski_genome_1999" class="csl-entry" role="doc-biblioentry">
Lenski, Richard E., Charles Ofria, Travis C. Collier, and Christoph Adami. 1999. <span>“Genome Complexity, Robustness and Genetic Interactions in Digital Organisms.”</span> <em>Nature</em> 400 (6745): 661–64. <a href="https://doi.org/10.1038/23245">https://doi.org/10.1038/23245</a>.
</div>
<div id="ref-lenski_evolutionary_2003" class="csl-entry" role="doc-biblioentry">
Lenski, Richard E., Charles Ofria, Robert T. Pennock, and Christoph Adami. 2003. <span>“The Evolutionary Origin of Complex Features.”</span> <em>Nature</em> 423 (6936): 139–44. <a href="https://doi.org/10.1038/nature01568">https://doi.org/10.1038/nature01568</a>.
</div>
<div id="ref-levis_evaluating_2016" class="csl-entry" role="doc-biblioentry">
Levis, Nicholas A., and David W. Pfennig. 2016. <span>“Evaluating <span>‘<span>Plasticity</span>-<span>First</span>’</span> <span>Evolution</span> in <span>Nature</span>: <span>Key</span> <span>Criteria</span> and <span>Empirical</span> <span>Approaches</span>.”</span> <em>Trends in Ecology &amp; Evolution</em> 31 (7): 563–74. <a href="https://doi.org/10.1016/j.tree.2016.03.012">https://doi.org/10.1016/j.tree.2016.03.012</a>.
</div>
<div id="ref-li_digital_2004" class="csl-entry" role="doc-biblioentry">
Li, Ye, and Claus O. Wilke. 2004. <span>“Digital <span>Evolution</span> in <span>Time</span>-<span>Dependent</span> <span>Fitness</span> <span>Landscapes</span>.”</span> <em>Artificial Life</em> 10 (2): 123–34. <a href="https://doi.org/10.1162/106454604773563559">https://doi.org/10.1162/106454604773563559</a>.
</div>
<div id="ref-nahum_improved_2017" class="csl-entry" role="doc-biblioentry">
Nahum, Joshua R., Jevin West, Benjamin M. Althouse, Luis Zaman, Charles Ofria, and Benjamin Kerr. 2017. <span>“Improved Adaptation in Exogenously and Endogenously Changing Environments.”</span> In <em>Proceedings of the 14th <span>European</span> <span>Conference</span> on <span>Artificial</span> <span>Life</span> <span>ECAL</span> 2017</em>, 306–13. Lyon, France: MIT Press. <a href="https://doi.org/10.7551/ecal_a_052">https://doi.org/10.7551/ecal_a_052</a>.
</div>
<div id="ref-ofria_avida:_2009" class="csl-entry" role="doc-biblioentry">
Ofria, Charles, David M. Bryson, and Claus O. Wilke. 2009. <span>“Avida: <span>A</span> <span>Software</span> <span>Platform</span> for <span>Research</span> in <span>Computational</span> <span>Evolutionary</span> <span>Biology</span>.”</span> In <em>Artificial <span>Life</span> <span>Models</span> in <span>Software</span></em>, edited by Maciej Komosinski and Andrew Adamatzky, 3–35. London: Springer London. <a href="https://doi.org/10.1007/978-1-84882-285-6_1">https://doi.org/10.1007/978-1-84882-285-6_1</a>.
</div>
<div id="ref-paaby_cryptic_2014" class="csl-entry" role="doc-biblioentry">
Paaby, Annalise B., and Matthew V. Rockman. 2014. <span>“Cryptic Genetic Variation: Evolution’s Hidden Substrate.”</span> <em>Nature Reviews Genetics</em> 15 (4): 247–58. <a href="https://doi.org/10.1038/nrg3688">https://doi.org/10.1038/nrg3688</a>.
</div>
<div id="ref-paenke_influence_2007" class="csl-entry" role="doc-biblioentry">
Paenke, Ingo, Bernhard Sendhoff, and Tadeusz J. Kawecki. 2007. <span>“Influence of <span>Plasticity</span> and <span>Learning</span> on <span>Evolution</span> Under <span>Directional</span> <span>Selection</span>.”</span> <em>The American Naturalist</em> 170 (2): E47–58. <a href="https://doi.org/10.1086/518952">https://doi.org/10.1086/518952</a>.
</div>
<div id="ref-pigliucci_phenotypic_2006" class="csl-entry" role="doc-biblioentry">
Pigliucci, M. 2006. <span>“Phenotypic Plasticity and Evolution by Genetic Assimilation.”</span> <em>Journal of Experimental Biology</em> 209 (12): 2362–67. <a href="https://doi.org/10.1242/jeb.02070">https://doi.org/10.1242/jeb.02070</a>.
</div>
<div id="ref-price_role_2003" class="csl-entry" role="doc-biblioentry">
Price, Trevor D., Anna Qvarnström, and Darren E. Irwin. 2003. <span>“The Role of Phenotypic Plasticity in Driving Genetic Evolution.”</span> <em>Proceedings of the Royal Society of London. Series B: Biological Sciences</em> 270 (1523): 1433–40. <a href="https://doi.org/10.1098/rspb.2003.2372">https://doi.org/10.1098/rspb.2003.2372</a>.
</div>
<div id="ref-rice_analyzing_1989" class="csl-entry" role="doc-biblioentry">
Rice, William R. 1989. <span>“Analyzing <span>Tables</span> of <span>Statistical</span> <span>Tests</span>.”</span> <em>Evolution</em> 43 (1): 223. <a href="https://doi.org/10.2307/2409177">https://doi.org/10.2307/2409177</a>.
</div>
<div id="ref-schaum_plasticity_2014" class="csl-entry" role="doc-biblioentry">
Schaum, C. Elisa, and Sinéad Collins. 2014. <span>“Plasticity Predicts Evolution in a Marine Alga.”</span> <em>Proceedings of the Royal Society B: Biological Sciences</em> 281 (1793): 20141486. <a href="https://doi.org/10.1098/rspb.2014.1486">https://doi.org/10.1098/rspb.2014.1486</a>.
</div>
<div id="ref-schlichting_hidden_2008" class="csl-entry" role="doc-biblioentry">
Schlichting, Carl D. 2008. <span>“Hidden <span>Reaction</span> <span>Norms</span>, <span>Cryptic</span> <span>Genetic</span> <span>Variation</span>, and <span>Evolvability</span>.”</span> <em>Annals of the New York Academy of Sciences</em> 1133 (1): 187–203. <a href="https://doi.org/10.1196/annals.1438.010">https://doi.org/10.1196/annals.1438.010</a>.
</div>
<div id="ref-schlichting_phenotypic_2014" class="csl-entry" role="doc-biblioentry">
Schlichting, Carl D., and Matthew A. Wund. 2014. <span>“Phenotypic <span>Plasticity</span> and <span>Epigenetic</span> <span>Marking</span>: <span>An</span> <span>Assessment</span> of <span>Evidence</span> for <span>Genetic</span> <span>Accommodation</span>.”</span> <em>Evolution</em> 68 (3): 656–72. <a href="https://doi.org/10.1111/evo.12348">https://doi.org/10.1111/evo.12348</a>.
</div>
<div id="ref-schwander_genes_2011" class="csl-entry" role="doc-biblioentry">
Schwander, Tanja, and Olof Leimar. 2011. <span>“Genes as Leaders and Followers in Evolution.”</span> <em>Trends in Ecology &amp; Evolution</em> 26 (3): 143–51. <a href="https://doi.org/10.1016/j.tree.2010.12.010">https://doi.org/10.1016/j.tree.2010.12.010</a>.
</div>
<div id="ref-smith_hitch-hiking_1974" class="csl-entry" role="doc-biblioentry">
Smith, John Maynard, and John Haigh. 1974. <span>“The Hitch-Hiking Effect of a Favourable Gene.”</span> <em>Genetical Research</em> 23 (1): 23–35. <a href="https://doi.org/10.1017/S0016672300014634">https://doi.org/10.1017/S0016672300014634</a>.
</div>
<div id="ref-van_den_bergh_experimental_2018" class="csl-entry" role="doc-biblioentry">
Van den Bergh, Bram, Toon Swings, Maarten Fauvart, and Jan Michiels. 2018. <span>“Experimental <span>Design</span>, <span>Population</span> <span>Dynamics</span>, and <span>Diversity</span> in <span>Microbial</span> <span>Experimental</span> <span>Evolution</span>.”</span> <em>Microbiology and Molecular Biology Reviews</em> 82 (3): e00008–18, /mmbr/82/3/e00008–18.atom. <a href="https://doi.org/10.1128/MMBR.00008-18">https://doi.org/10.1128/MMBR.00008-18</a>.
</div>
<div id="ref-wennersten_population-level_2012" class="csl-entry" role="doc-biblioentry">
Wennersten, Lena, and Anders Forsman. 2012. <span>“Population-Level Consequences of Polymorphism, Plasticity and Randomized Phenotype Switching: A Review of Predictions.”</span> <em>Biological Reviews</em> 87 (3): 756–67. <a href="https://doi.org/10.1111/j.1469-185X.2012.00231.x">https://doi.org/10.1111/j.1469-185X.2012.00231.x</a>.
</div>
<div id="ref-west-eberhard_developmental_2005" class="csl-entry" role="doc-biblioentry">
West-Eberhard, M. J. 2005. <span>“Developmental Plasticity and the Origin of Species Differences.”</span> <em>Proceedings of the National Academy of Sciences</em> 102 (Supplement 1): 6543–49. <a href="https://doi.org/10.1073/pnas.0501844102">https://doi.org/10.1073/pnas.0501844102</a>.
</div>
<div id="ref-west-eberhard_developmental_2003" class="csl-entry" role="doc-biblioentry">
West-Eberhard, May Jane. 2003. <em>Developmental <span>Plasticity</span> and <span>Evolution</span>.</em> Oxford University Press. <a href="http://search.ebscohost.com.proxy1.cl.msu.edu/login.aspx?direct=true&amp;db=e000xna&amp;AN=688592&amp;site=ehost-live">http://search.ebscohost.com.proxy1.cl.msu.edu/login.aspx?direct=true&amp;db=e000xna&amp;AN=688592&amp;site=ehost-live</a>.
</div>
<div id="ref-kotz_individual_1992" class="csl-entry" role="doc-biblioentry">
Wilcoxon, Frank. 1992. <span>“Individual <span>Comparisons</span> by <span>Ranking</span> <span>Methods</span>.”</span> In <em>Breakthroughs in <span>Statistics</span></em>, edited by Samuel Kotz and Norman L. Johnson, 196–202. New York, NY: Springer New York. <a href="https://doi.org/10.1007/978-1-4612-4380-9_16">https://doi.org/10.1007/978-1-4612-4380-9_16</a>.
</div>
<div id="ref-wilke_biology_2002" class="csl-entry" role="doc-biblioentry">
Wilke, Claus O., and Christoph Adami. 2002. <span>“The Biology of Digital Organisms.”</span> <em>Trends in Ecology &amp; Evolution</em> 17 (11): 528–32. <a href="https://doi.org/10.1016/S0169-5347(02)02612-5">https://doi.org/10.1016/S0169-5347(02)02612-5</a>.
</div>
<div id="ref-wilke_evolution_2001" class="csl-entry" role="doc-biblioentry">
Wilke, Claus O., Jia Lan Wang, Charles Ofria, Richard E. Lenski, and Christoph Adami. 2001. <span>“Evolution of Digital Organisms at High Mutation Rates Leads to Survival of the Flattest.”</span> <em>Nature</em> 412 (6844): 331–33. <a href="https://doi.org/10.1038/35085569">https://doi.org/10.1038/35085569</a>.
</div>
<div id="ref-winger_long_2019" class="csl-entry" role="doc-biblioentry">
Winger, Benjamin M., Giorgia G. Auteri, Teresa M. Pegan, and Brian C. Weeks. 2019. <span>“A Long Winter for the <span>Red</span> <span>Queen</span>: Rethinking the Evolution of Seasonal Migration.”</span> <em>Biological Reviews</em> 94 (3): 737–52. <a href="https://doi.org/10.1111/brv.12476">https://doi.org/10.1111/brv.12476</a>.
</div>
<div id="ref-wund_assessing_2012" class="csl-entry" role="doc-biblioentry">
Wund, M. A. 2012. <span>“Assessing the <span>Impacts</span> of <span>Phenotypic</span> <span>Plasticity</span> on <span>Evolution</span>.”</span> <em>Integrative and Comparative Biology</em> 52 (1): 5–15. <a href="https://doi.org/10.1093/icb/ics050">https://doi.org/10.1093/icb/ics050</a>.
</div>
<div id="ref-zheng_cryptic_2019" class="csl-entry" role="doc-biblioentry">
Zheng, Jia, Joshua L. Payne, and Andreas Wagner. 2019. <span>“Cryptic Genetic Variation Accelerates Evolution by Opening Access to Diverse Adaptive Peaks.”</span> <em>Science</em> 365 (6451): 347–53. <a href="https://doi.org/10.1126/science.aax1837">https://doi.org/10.1126/science.aax1837</a>.
</div>
<div id="ref-zimmer_evolution_2013" class="csl-entry" role="doc-biblioentry">
Zimmer, Carl, and Douglas John Emlen. 2013. <em>Evolution: Making Sense of Life</em>. Greenwood Village, CO: Roberts; Company Publishers.
</div>
</div>
<section class="footnotes" role="doc-endnotes">
<hr />
<ol>
<li id="fn1" role="doc-endnote"><p>We repeated our experiments without genome size restrictions and observed qualitatively similar results (see supplemental material, <span class="citation" data-cites="supplemental_material">(<strong>supplemental_material?</strong>)</span>).<a href="#fnref1" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn2" role="doc-endnote"><p>One update in Avida is the amount of time required for the average organism to execute 30 instructions. See <span class="citation" data-cites="ofria_avida:_2009">(Ofria, Bryson, and Wilke 2009)</span> for more details.<a href="#fnref2" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
</ol>
</section>
</body>
</html>
