<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <meta name="author" content="First Author ^{1,*}, Co-Author ^{2} and Co-Author ^{1,2}" />
  <title>Phenotypic plasticity can promote the evolution and maintenance of complex features in fluctuating environments</title>
  <style>
    html {
      line-height: 1.7;
      font-family: Georgia, serif;
      font-size: 20px;
      color: #1a1a1a;
      background-color: #fdfdfd;
    }
    body {
      margin: 0 auto;
      max-width: 40em;
      padding-left: 50px;
      padding-right: 50px;
      padding-top: 50px;
      padding-bottom: 50px;
      hyphens: auto;
      word-wrap: break-word;
      text-rendering: optimizeLegibility;
      font-kerning: normal;
    }
    @media (max-width: 600px) {
      body {
        font-size: 0.9em;
        padding: 1em;
      }
    }
    @media print {
      body {
        background-color: transparent;
        color: black;
      }
      p, h2, h3 {
        orphans: 3;
        widows: 3;
      }
      h2, h3, h4 {
        page-break-after: avoid;
      }
    }
    p {
      margin-top: 1.7em;
    }
    a {
      color: #1a1a1a;
    }
    a:visited {
      color: #1a1a1a;
    }
    img {
      max-width: 100%;
    }
    h1, h2, h3, h4, h5, h6 {
      margin-top: 1.7em;
    }
    ol, ul {
      padding-left: 1.7em;
      margin-top: 1.7em;
    }
    li > ol, li > ul {
      margin-top: 0;
    }
    blockquote {
      margin: 1.7em 0 1.7em 1.7em;
      padding-left: 1em;
      border-left: 2px solid #e6e6e6;
      font-style: italic;
    }
    code {
      font-family: Menlo, Monaco, 'Lucida Console', Consolas, monospace;
      background-color: #f0f0f0;
      font-size: 85%;
      margin: 0;
      padding: .2em .4em;
    }
    pre {
      line-height: 1.5em;
      padding: 1em;
      background-color: #f0f0f0;
      overflow: auto;
    }
    pre code {
      padding: 0;
      overflow: visible;
    }
    hr {
      background-color: #1a1a1a;
      border: none;
      height: 1px;
      margin-top: 1.7em;
    }
    table {
      border-collapse: collapse;
      width: 100%;
      overflow-x: auto;
      display: block;
    }
    th, td {
      border-bottom: 1px solid lightgray;
      padding: 1em 3em 1em 0;
    }
    header {
      margin-bottom: 6em;
      text-align: center;
    }
    nav a:not(:hover) {
      text-decoration: none;
    }
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    span.underline{text-decoration: underline;}
    div.column{display: inline-block; vertical-align: top; width: 50%;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    .display.math{display: block; text-align: center; margin: 0.5rem auto;}
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
    }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
</head>
<body>
<header id="title-block-header">
<h1 class="title">Phenotypic plasticity can promote the evolution and maintenance of complex features in fluctuating environments</h1>
<p class="author">First Author <span class="math inline"><sup>1, *</sup></span>, Co-Author <span class="math inline"><sup>2</sup></span> and Co-Author <span class="math inline"><sup>1, 2</sup></span></p>
</header>
<h1 id="introduction">Introduction</h1>
<p>Fluctuating environmental conditions are ubiquitous in natural systems. Organisms have evolved a wide range of evolved strategies for coping with environmental change, such as phenotypic plasticity <span class="citation" data-cites="ghalambor_adaptive_2007">(C. K. Ghalambor et al. 2007)</span>, bet hedging <span class="citation" data-cites="beaumont_experimental_2009">(Beaumont et al. 2009)</span>, periodic migration <span class="citation" data-cites="winger_long_2019">(Winger et al. 2019)</span>, and adaptive tracking <span class="citation" data-cites="barrett_adaptation_2008">(Barrett and Schluter 2008)</span>. The particular coping mechanisms that evolve in fluctuating environments shift the course of subsequent evolution <span class="citation" data-cites="wennersten_population-level_2012 schaum_plasticity_2014">(Wennersten and Forsman 2012; Schaum and Collins 2014)</span>. Identifying the mechanisms most likely to evolve and examining both the evolutionary constraints and opportunities associated with each is critical for us to understand and predict evolutionary outcomes in changing environments.</p>
<p>In this work, we focus on phenotypic plasticity, which can be defined as the capacity for a single genotype to alter phenotypic expression in response to a change in its environment <span class="citation" data-cites="west-eberhard_developmental_2003">(May Jane West-Eberhard 2003)</span>. Phenotypic plasticity is controlled by genes whose expression is coupled to one or more abiotic or biotic environmental signals. For example, the sex ratio of the crustacean <em>Gammarus duebeni</em> is modulated by changes in photoperiod and temperature <span class="citation" data-cites="dunn_two_2005">(Dunn et al. 2005)</span>, and the reproductive output of some invertebrate species is heightened when infected with parasites to compensate for offspring loss <span class="citation" data-cites="chadwick_parasite-mediated_2005">(Chadwick and Little 2005)</span>. In this study, we conducted digital evolution experiments to investigate how the evolution of adaptive phenotypic plasticity shifts the course of evolution in a cyclically changing environment. Specifically, we examined the effects of adaptive plasticity on subsequent genomic and phenotypic change, the capacity to evolve and then maintain novel traits, and the accumulation of deleterious alleles.</p>
<p>Evolutionary biologists have long been interested in how evolutionary change across generations is influenced by phenotypic plasticity because of its role in generating phenotypic variance <span class="citation" data-cites="gibert_phenotypic_2019">(Gibert, Debat, and Ghalambor 2019)</span>. The effects of phenotypic plasticity on adaptive evolution have been disputed, as few studies have been able to observe both the initial patterns of plasticity and the subsequent divergence of traits in natural populations <span class="citation" data-cites="ghalambor_adaptive_2007 wund_assessing_2012 forsman_rethinking_2015 ghalambor_non-adaptive_2015 hendry_key_2016">(C. K. Ghalambor et al. 2007; Wund 2012; Forsman 2015; Cameron K. Ghalambor et al. 2015; Hendry 2016)</span>. In a changing environment, adaptive phenotypic plasticity provides a mechanism for organisms to regulate trait expression within their lifetime, which can stabilize populations through changes <span class="citation" data-cites="gibert_phenotypic_2019">(Gibert, Debat, and Ghalambor 2019)</span> [conceptual figure (see comment)?]. In this context, the stabilization effects of adaptive plasticity have been hypothesized to constrain the rate of adaptive evolution <span class="citation" data-cites="gupta_study_1982 ancel_undermining_2000 huey_behavioral_2003 price_role_2003 paenke_influence_2007">(Gupta and Lewontin 1982; Ancel 2000; Huey, Hertz, and Sinervo 2003; Price, Qvarnström, and Irwin 2003; Paenke, Sendhoff, and Kawecki 2007)</span>. That is, directional selection may be weak if environmentally-induced phenotypes are close to the optimum; as such, adaptively plastic populations may evolve slowly (relative to non-plastic populations) unless there is a substantial fitness cost to plasticity.</p>
<figure>
<embed src="media/reaction-norms.pdf" id="fig:reaction-norms" /><figcaption aria-hidden="true"> <strong>todo.</strong> todo. </figcaption>
</figure>
<p>Phenotypic plasticity allows for the accumulation of genetic variation in genomic regions that are unexpressed under current environmental conditions. Such cryptic (“hidden”) genetic variation can serve as a source of diversity in the population, upon which selection can act when the environment changes <span class="citation" data-cites="schlichting_hidden_2008 levis_evaluating_2016">(Schlichting 2008; Levis and Pfennig 2016)</span>. It remains unclear to what extent and under what circumstances this cryptic variation caches adaptive potential or merely accumulates deleterious alleles <span class="citation" data-cites="gibson_uncovering_2004 paaby_cryptic_2014 zheng_cryptic_2019">(Gibson and Dworkin 2004; Paaby and Rockman 2014; Zheng, Payne, and Wagner 2019)</span>.</p>
<p>The “genes as followers” hypothesis (also known as the “plasticity first” hypothesis) predicts that phenotypic plasticity may facilitate adaptive evolutionary change by producing variants with enhanced fitness under stressful or novel conditions <span class="citation" data-cites="west-eberhard_developmental_2003 schwander_genes_2011 levis_evaluating_2016">(May Jane West-Eberhard 2003; Schwander and Leimar 2011; Levis and Pfennig 2016)</span>. Environmentally-induced trait changes can be refined through selection over time (i.e., genetic accommodation). Further, selection may drive plastic phenotypes to lose their environmental dependence over time in a process known as genetic assimilation <span class="citation" data-cites="west-eberhard_developmental_2005 pigliucci_phenotypic_2006 crispo_baldwin_2007 schlichting_phenotypic_2014 levis_evaluating_2016">(M. J. West-Eberhard 2005; Pigliucci 2006; Crispo 2007; Schlichting and Wund 2014; Levis and Pfennig 2016)</span>. In this way, environmentally-induced phenotypic changes can precede an evolutionary response.</p>
<p>Phenotypic plasticity may also ‘rescue’ populations from extinction under changing environmental conditions by buffering populations against novel stressors. This buffer promotes stability and persistence and grants populations time to further adapt to rapidly changing environmental conditions <span class="citation" data-cites="west-eberhard_developmental_2003 chevin_when_2010">(May Jane West-Eberhard 2003; Chevin and Lande 2010)</span>. Disparate predictions about how phenotypic plasticity may shift the course of subsequent evolution are not necessarily mutually exclusive. Genetic and environmental contexts determine if and to what extent phenotypic plasticity promotes or constrains subsequent evolution.</p>
<p>Experimental studies investigating the relationship between phenotypic plasticity and evolutionary outcomes can be challenging to conduct in natural systems. Such experiments would require the ability to irreversibly toggle plasticity followed by long periods of evolution during which detailed phenotypic data would need to be collected. Digital evolution experiments have emerged as a powerful research framework from which evolution can be studied. In digital evolution, self-replicating computer programs (digital organisms) compete for resources, mutate, and evolve following Darwinian dynamics <span class="citation" data-cites="mckinley_harnessing_2008">(McKinley et al. 2008)</span>. Digital evolution studies balance the speed and transparency of mathematical and computational simulations with the open-ended realism of laboratory experiments. Modern computers allow us to observe many generations of digital evolution at tractable time scales; thousands of generations can take mere minutes as opposed to months, years, or centuries. Digital evolution systems also allow for perfect, non-invasive data tracking. Such transparency permits the tracking of complete evolutionary histories within an experiment, which circumvents the historical problem of drawing evolutionary inferences using incomplete records (from frozen samples or even fossils) and extant genetic sequences. Additionally, digital evolution systems allow for experimental manipulations and analyses that go beyond what is possible in wet-lab experiments. Such analyses have included exhaustive knockouts of every loci to identify the functionality of each <span class="citation" data-cites="lenski_evolutionary_2003">(Lenski et al. 2003)</span>, comprehensive characterization of local mutational landscapes <span class="citation" data-cites="lenski_genome_1999 canino-koning_fluctuating_2019">(Lenski et al. 1999; Canino-Koning, Wiser, and Ofria 2019)</span>, and the real-time reversion of all deleterious mutations as they occur to isolate their long-term effects on evolutionary outcomes <span class="citation" data-cites="covert_experiments_2013">(Covert et al. 2013)</span>. Digital evolution studies allow us to directly toggle the possibility for adaptive plastic responses to evolve, which enables us to empirically test hypotheses that were previously relegated to theoretical analyses.</p>
<p>In this work, we use the Avida Digital Evolution Platform <span class="citation" data-cites="ofria_avida:_2009">(Ofria, Bryson, and Wilke 2009)</span>. Avida is an open-source system that has been used to conduct a wide range of well-regarded studies on evolutionary dynamics, including the origins of complex features <span class="citation" data-cites="lenski_evolutionary_2003">(Lenski et al. 2003)</span>, the survival of the flattest effect <span class="citation" data-cites="wilke_evolution_2001">(Wilke et al. 2001)</span>, and the origins of reproductive division of labor <span class="citation" data-cites="goldsby_evolutionary_2014">(Goldsby et al. 2014)</span>. Our experiments build directly on previous studies in Avida that characterized the <em>de novo</em> evolution of adaptive phenotypic plasticity <span class="citation" data-cites="clune_investigating_2007 lalejini_evolutionary_2016">(Clune, Ofria, and Pennock 2007; Lalejini and Ofria 2016)</span> and studies that characterized the evolutionary consequences of fluctuating environments for populations of non-plastic digital organisms <span class="citation" data-cites="wilke_evolution_2001 canino-koning_fluctuating_2019">(Wilke et al. 2001; Canino-Koning, Wiser, and Ofria 2019)</span>. Of particular relevance, <span class="citation" data-cites="clune_investigating_2007">(Clune, Ofria, and Pennock 2007)</span> and <span class="citation" data-cites="lalejini_evolutionary_2016">(Lalejini and Ofria 2016)</span> experimentally demonstrated that adaptive phenotypic plasticity can evolve given the following four conditions (as identified by <span class="citation" data-cites="ghalambor_behavior_2010">(Cameron K. Ghalambor, Angeloni, and Carroll 2010)</span>): (1) populations experience temporal environmental variation, (2) these environments are differentiable by reliable cues, (3) each environment favors different phenotypic traits, and (4) no single phenotype exhibits high fitness across all environments. We build on this previous work but shift our focus from the evolutionary causes of adaptive phenotypic plasticity to investigate its evolutionary consequences in a fluctuating environment.</p>
<p>Each of our experiments are divided into two phases: in phase one, we precondition sets of founder organisms with differing plastic or non-plastic adaptations; in phase two, we examine the subsequent evolution of populations founded with organisms from phase one under specific environmental conditions [figure]. First, we examine the evolutionary histories of phase two populations to test whether adaptive plasticity constrained subsequent genomic and phenotypic changes. Next, we evaluate how adaptive plasticity influences fitness landscape exploration and exploitation by identifying how well populations produced by each type of founder are able to evolve and retain novel adaptive traits. Finally, we test whether adaptive plasticity facilitated the accumulation of deleterious genes via cryptic genetic variation by measuring the frequency with which deleterious mutations occur along successful lineages.</p>
<p>We found that the evolution of adaptive plasticity reduced subsequent rates of evolutionary change in a cyclic environment. The non-plastic populations underwent more frequent selective sweeps and accumulated many more genetic changes over time, as non-plastic populations relied on genetic variation from <em>de novo</em> mutations to continuously re-adapt to environmental changes. We found that the evolution of adaptive phenotypic plasticity buffers populations against environmental fluctuations, whereas repeated selective sweeps in non-plastic populations drive the accumulation of deleterious genes and the loss of secondary beneficial traits via deleterious hitchhiking. As such, adaptively plastic populations were better able to retain novel traits than their non-plastic counterparts.</p>
<h1 id="materials-and-methods">Materials and Methods</h1>
<h2 id="the-avida-digital-evolution-platform">The Avida Digital Evolution Platform</h2>
<p>Avida is a study system wherein self-replicating computer programs (digital organisms) compete for space on a finite toroidal grid <span class="citation" data-cites="ofria_avida:_2009">(Ofria, Bryson, and Wilke 2009)</span>. Each digital organism is defined by a linear sequence of program instructions (its genome) and a set of virtual hardware components used to interpret and express those instructions. Genomes are expressed sequentially except when the execution of one instruction deterministically changes which instruction should be executed next (e.g., a ‘jump’ instruction). Genomes are built using an instruction set that is both robust (<em>i.e.</em>, any ordering of instructions is syntactically valid, though not necessarily meaningful) and Turing Complete (<em>i.e.</em>, able to represent any computable function, though not necessarily in an efficient manner) [cite]. The instruction set includes operations for basic computations, flow control (e.g., conditional logic and looping), input, output, and self-replication.</p>
<p>Organisms in Avida reproduce asexually by copying their genome instruction-by-instruction and then dividing. However, copy operations are imperfect and can result in single-instruction substitution mutations in an offspring’s genome. For this work, we configured copy operations to err at a rate of one expected mutation for every 400 instructions copied (i.e, a per-instruction error rate of 0.0025). We held individual genomes at a fixed length of 100 instructions; that is, we did not include insertion and deletion mutations. We used fixed-length genomes to control for treatment-specific conditions resulting in the evolution of substantially different genome sizes [supplement, citations]<a href="#fn1" class="footnote-ref" id="fnref1" role="doc-noteref"><sup>1</sup></a>, which could, on its own, drive differences in evolutionary outcomes among experimental treatments. When an organism divides in Avida, its offspring is placed in a random location on the toroidal grid, replacing any previous occupant. For this work, we used the default 60 by 60 grid size, which limits the maximum population size to 3600 organisms. As such, improvements to the speed of self-replication are advantageous in the competition for space.</p>
<p>During evolution, organism replication rates improve in two ways: by improving genome efficiency (e.g., using a more compact encoding) or by accelerating the rate at which the genome is expressed (their “metabolic rate”). An organism’s metabolic rate determines the speed at which it executes instructions in its genome. Initially, an organism’s metabolic rate is proportional to the length of its genome, but that rate is adjusted as it completes designated tasks, such as performing Boolean logic computations <span class="citation" data-cites="ofria_avida:_2009">(Ofria, Bryson, and Wilke 2009)</span>. In this way, we can reward or punish particular phenotypic traits.</p>
<h3 id="phenotypic-plasticity-in-avida">Phenotypic plasticity in Avida</h3>
<p>In this work, a digital organism’s phenotype is the set of Boolean logic functions that it performs in a given environment. Sensory instructions in the Avida instruction set allow organisms to detect how performing a particular logic function would affect their metabolic rate [cite - supplement]. We defined a phenotypically plastic organism as one that uses sensory information to alter which logic functions it performs based on the environment.</p>
<p>Phenotypic plasticity in Avida can be adaptive or non-adaptive. Adaptive plasticity shifts net task expression closer to the optimum for a given set of environments. Non-adaptive plasticity either changes task expression in a neutral way or further from the optimum for a given set of environments.</p>
<h2 id="sec:methods:experiment">Experimental design</h2>
<p>We conducted three independent experiments using Avida to investigate how the evolution of adaptive plasticity influences evolutionary outcomes in fluctuating environments. For each experiment, we compared the evolutionary outcomes of populations evolved under three treatments: (1) a <strong>PLASTIC</strong> treatment where the environment fluctuates, and digital organisms can differentiate between environmental states using sensory instructions; (2) a <strong>NON-PLASTIC</strong> treatment where the environment fluctuates, but sensory instructions are disabled; and (3) a <strong>STATIC</strong> control where organisms evolve in a constant environment.</p>
<p>Each experiments was divided into two phases that each lasted for 200,000 updates<a href="#fn2" class="footnote-ref" id="fnref2" role="doc-noteref"><sup>2</sup></a> of evolution (Figure [XX]), which is approximately 30,000 to 40,000 generations. In phase one of each experiment, we preconditioned populations to their treatment-specific conditions. Each experiment in our study shares a common setup for phase one (described in Section <a href="#sec:methods:experiment:phase-one" data-reference-type="ref" data-reference="sec:methods:experiment:phase-one">3.2.2</a>).</p>
<p>In phase two, we examined the subsequent evolution of populations founded with organisms from phase one under given treatment conditions. [Phase two differed among each of our experiments (described in Sections [X,Y,and Z]), depending the goal of the particular experiment.] During phase two, we tracked each population’s evolutionary history as well as saving the full final population. All comparisons between treatments were performed on data from populations during phase two.</p>
<h3 id="sec:methods:experiment:environments">Environments</h3>
<p>We constructed three experimental environments, abbreviated hereafter as “ENV-A,” “ENV-B,” and “ENV-ALL” ([Figure XX]). In ENV-A, organisms are rewarded for performing the NOT, AND, and OR Boolean logic tasks, but are punished for performing the NAND, OR-NOT, and AND-NOT logic tasks. Conversely in ENV-B, organisms are rewarded for performing NAND, OR-NOT, and AND-NOT, but are punished for performing NOT, AND, and OR. Finally, in ENV-ALL, all six tasks are rewarded and none are punished. Each rewarded task performed by an organism doubles their metabolic rate (allowing them to execute twice as many instructions in the same amount of time), and each punished task performed halves their metabolic rate.</p>
<p>In both the PLASTIC and NON-PLASTIC conditions, the environment cycles between equal-length periods of ENV-A and ENV-B. Each of these periods persist for 100 updates (approximately 15 to 20 generations). Thus, populations experience a total of 1,000 full periods of ENV-A and 1,000 full periods of ENV-B during each experiment phase.</p>
<p>Organisms can differentiate ENV-A and ENV-B by executing one of six sensory instructions, each associated with a particular logical task (NOT, NAND, AND, OR-NOT, OR, and AND-NOT); these sensory instructions detect whether their associated task is currently rewarded or punished (see Supplemental Section [blah] for details [cite]). By using sensory information in combination with execution flow-control instructions, organisms can conditionally perform different logic tasks depending on the current environmental conditions.</p>
<h3 id="sec:methods:experiment:phase-one">Experiment Phase 1 – Environment preconditioning</h3>
<p>At the beginning of phase one for each experiment, we founded 100 independent populations for each of the PLASTIC, NON-PLASTIC, and STATIC treatments. Each phase-one population was founded with a common ancestral strain capable only of self-replication [cite - supplement]. Phase one gives populations time (200,000 updates) to adapt to their treatment conditions, affording adaptive phenotypic plasticity the opportunity to evolve <em>de novo</em> in the PLASTIC treatment. At the end of phase one, we extract the most abundant (i.e., dominant) genotype from each replicate population to found a new population for phase two.</p>
<p>The evolution of adaptive phenotypic plasticity is not a guaranteed outcome in phase one of the PLASTIC treatment. For the PLASTIC treatment, we only continue to phase two if the dominant genotype is “optimally plastic.” An optimally plastic genotype expresses each of the rewarded tasks in a given environment and none of the punished tasks in that environment. We test for optimal plasticity by expressing a given genotype [independently/separately] in ENV-A and ENV-B and analyzing its phenotype in each environment. This approach ensures that measurements taken on PLASTIC-treatment populations during the second phase of each experiment are representative of populations with adaptive phenotypic plasticity.</p>
<h3 id="experiment-phase-2a-evolutionary-change-rate">Experiment Phase 2A – Evolutionary change rate</h3>
<p>[1 sentence of motivation/hypothesis?]. Phase two continued exactly as phase one, except we tracked the rates of evolutionary change in each of the PLASTIC-, NON-PLASTIC-, and STATIC-treatment populations. Specifically, we quantified evolutionary change rates using four metrics (defined in Section [X]): (1) number of coalescence events, (2) mutation accumulation, (3) phenotypic volatility, and (4) mutational stability. We additionally examined how the genetic architectures of organisms and their ancestors changed over time.</p>
<h3 id="experiment-phase-2b-novel-task-evolution">Experiment Phase 2B – Novel task evolution</h3>
<p>[1 sentence of motivation/hypothesis?]. Phase two continued exactly as phase one, except we used the expanded task set of 77 Boolean logic tasks during the second phase of evolution <span class="citation" data-cites="ofria_avida:_2009">(Ofria, Bryson, and Wilke 2009)</span>. This task set includes the six phase one tasks (NOT, NAND, AND, OR-NOT, OR, and AND-NOT; hereafter called “base” tasks) plus 71 new phase two tasks (hereafter called “novel” tasks). Under all experimental treatments, organisms could improve their metabolic rate by performing any of the 71 novel tasks. The six base tasks were still present in the environment and continued to be rewarded or punished according to the particular treatment conditions. As such, in fluctuating environments, the six base tasks continued to fluctuate, but the additional 71 tasks were always rewarded; in static environments, performing any of the 77 logic tasks was always beneficial. During this experiment, we tracked the extent to which populations evolving under each treatment were capable of acquiring and retaining novel tasks.</p>
<p>An organism received a 10% metabolic rate improvement for each of the novel tasks it performed (limited to one reward per unique task). This reward provided a selective pressure to evolve these tasks, but their benefits did not overwhelm existing treatment-specific selective pressures. As such, populations in the PLASTIC and NON-PLASTIC treatments could not easily escape environmental fluctuations by abandoning the fluctuating base tasks.</p>
<h3 id="experiment-phase-2c-deleterious-instruction-accumulation">Experiment Phase 2C – Deleterious instruction accumulation</h3>
<p>[In this experiment,] we tested the propensity for deleterious instructions to accumulate in genomes under each of the PLASTIC, NON-PLASTIC, and STATIC treatments. Phase two continued exactly as phase one, except we added a <code>poison</code> instruction to the instruction set during phase two. When executed by an organism, the <code>poison</code> instruction reduces the organism’s metabolic rate (and thus reproductive success), but does not otherwise alter the organism’s function. We imposed a 10% penalty each time an organism executed the <code>poison</code> instruction, making the instruct explicitly deleterious (see [supplementary material] for tests with 3% and 30% penalties, which produced consistent experimental results). [We measured/tracked the number of times a <code>poison</code> instruction is executed by each genotype along the dominant lineage (including the final dominant genotype).]</p>
<h3 id="measurements-and-experimental-analyses">Measurements and experimental analyses</h3>
<p>For each of our experiments, we tracked and analyzed the phylogenetic histories of evolving populations during phase two. For each replicate, we isolated a representative lineage from its founding organism to a member of the dominant (i.e., most abundant) genotype at the end of its evolution. Because our experimental treatments do not support long-term coexistence, each of these lineages represented the majority of evolutionary history from a given population at the end of our experiment.</p>
<p>We quantified rates of evolutionary change using the following four metrics: (1) number of <strong>coalescence events</strong> that have occurred, which indicates the frequency of selective sweeps in the population; (2) <strong>mutation accumulation</strong>, which is the sum of all mutations that have occurred along a lineage; (3) <strong>phenotypic volatility</strong>, which is the number of instances where parent and offspring phenotypes do not match along a lineage, as measured under a given condition; and (4) <strong>mutational stability</strong>, which is the proportion of <em>mutated</em> offspring along a lineage whose phenotypes do not match that of their parent, as measured under a given condition.</p>
<p>To calculate phenotypic volatility for a given lineage, we expressed (i.e., evaluated) each genotype along that lineage in a treatment-specific condition, and we summed the number of changes in task profiles between consecutive genotypes. For lineages evolved in environments fluctuating between ENV-A and ENV-B, we evaluate genotypes in both environmental conditions and count only changes in its <em>aggregate</em> phenotype; this technique ensures that environmentally-induced changes are excluded from our measurement. Phenotypic volatility as defined here illuminates the rate at which accumulated genetic changes actually change the phenotype along a lineage.</p>
<p>We measured mutational stability as the fraction of mutated offspring along a given lineage with a different phenotype than their parent. For lineages evolved in fluctuating environments, we evaluated mutants under both ENV-A and ENV-B and counted all changes in the <em>aggregate</em> task profile; like our measure of phenotypic volatility, this technique ensures that environmentally-induced changes are excluded from our measurement. Mutational stability examines the frequency at which mutations effect changes in an offspring’s phenotype.</p>
<p>In experiments that introduce novel tasks during phase two, we measured task discovery, task performance, and task loss along representative lineages. <strong>Task discovery</strong> measures a given lineage’s level of <em>exploration</em> of the fitness landscape (i.e., the mapping between genetic space and phenotype space) <span class="citation" data-cites="canino-koning_fluctuating_2019">(Canino-Koning, Wiser, and Ofria 2019)</span>. We calculated task discovery as the total number of unique tasks ever performed along the lineage, even if a task is later lost; as such, a lineage’s task discovery measurement ranged from 0 to 71.</p>
<p><strong>Task performance</strong> measures the level of <em>exploitation</em> of the fitness landscape at a given point in time. In this work, we summarized task performance using a count of unique tasks completed by a representative organism from each population. We focused on an organism from the dominant genotype at the end of the experiment as the most representative phenotype in the evolved population.</p>
<p><strong>Task loss</strong> measures how often a lineage fails to retain evolved traits over time and thus indicates the ability for traits to be maintained over time. We calculated task loss as the number of times along a lineage that a task is performed by a parent but not its offspring.</p>
<p>For an individual organism, we can perform <strong>knockout experiments</strong> to identify which instructions are responsible for producing a given phenotypic outcome. To perform a knockout, we duplicate the organism, replacing a single instruction with an inert “no-operation” instruction. We then identify any phenotypic changes by contrasting the execution results of the “knockout” organism and the original. Such changes provide evidence of the role that the original instruction must have played in the genome. For example, when an organism performs the NAND task but loses it when an instruction is knocked out, we categorize that instruction as part of the NAND task machinery. We use knockout experiments to characterize the role of each instruction in the genomes of every organism along all study lineages, revealing how genetic architectures change over time.</p>
<h2 id="statistical-analyses-and-software-availability">Statistical analyses and software availability</h2>
<p>Across all of our experiments, we differentiated between sample distributions using non-parametric statistical tests. For each major analysis, we first performed a Kruskal-Wallis test <span class="citation" data-cites="kruskal_use_1952">(Kruskal and Wallis 1952)</span> to determine if there were significant differences in results from the PLASTIC, NON-PLASTIC, and STATIC treatments. If so, we applied a Wilcoxon rank-sum test <span class="citation" data-cites="kotz_individual_1992">(Wilcoxon 1992)</span> to distinguish between pairs of treatments. We applied Bonferroni corrections for multiple comparisons <span class="citation" data-cites="rice_analyzing_1989">(Rice 1989)</span> where appropriate.</p>
<p>We conducted our experiments using a modified version of the Avida software, which is open source and freely available on GitHub [cite]. We used Python [cite] for data processing, and we conducted all statistical analyses using R version [x] [cite]. We used the tidyverse collection of R packages [cite] to wrangle data, and we used the following R packages for graphing and visualization: [list and citations]. We used R markdown [cite] and bookdown [cite] to generate web-enabled supplemental material. All of the source code for our experiments and analyses, including guides for replication and configuration files, can be found in our supplemental material [cite]. Additionally, our experimental data is available on the Open Science framework at [url] [cite].</p>
<h1 id="results">Results</h1>
<h2 id="evolutionary-change">Evolutionary change</h2>
<figure>
<embed src="media/evolutionary-change-full-panel.pdf" id="fig:evolutionary-dynamics" style="width:100.0%" /><figcaption aria-hidden="true"> <strong>todo title.</strong> todo. </figcaption>
</figure>
<figure>
<embed src="media/locus_slice_combined.pdf" id="fig:architecture_locus_functionality" style="width:100.0%" /><figcaption aria-hidden="true"> <strong>todo title.</strong> Representative samples from each condition at the end of phase two. The 100-loci genome of each sample is shown, with colors indicating what functionality a locus has for each of the six base tasks. </figcaption>
</figure>
<p>[ ]</p>
<p>Adaptive phenotypic plasticity evolved in 42 of 100 replicates from the PLASTIC treatment during phase one of Experiment I; we used this more limited group to seed the 42 phase-two PLASTIC replicates. Figure [x] gives the number of coalescence events, mutation accumulation, and phenotypic volatility for PLASTIC, NON-PLASTIC, and STATIC treatments. According to each of these metrics, NON-PLASTIC populations experienced a larger magnitude of evolutionary change than either PLASTIC or STATIC populations. Specifically, we observed significantly more coalescence events in NON-PLASTIC populations than in PLASTIC or STATIC populations [stats]. NON-PLASTIC lineages had significantly higher mutation accumulation and phenotypic volatility than PLASTIC or STATIC lineages [stats]. Additionally, we found that significantly more generations of evolution elapsed in NON-PLASTIC populations than in PLASTIC or STATIC populations [stats]; that is, NON-PLASTIC populations exhibited a significantly higher rate of generational turnover ([data - mean, +- std dev]) than PLASTIC ([data - mean, +- stddev]) or STATIC ([data - mean, +- stddev]) populations.</p>
<p>Next, we examined the <em>rates</em> of evolutionary change across treatments to determine if the larger magnitude of evolutionary change in NON-PLASTIC populations is due to the number of additional generations elapsed during the experiment. Specifically, we calculated the average number of generations between coalescence events as well as both genotypic fidelity and phenotypic fidelity (i.e., the frequency that an offspring’s genotype or phenotype is identical to that of its parent along the dominant lineage). We found that significantly fewer generations elapse between coalescence events in NON-PLASTIC populations than in either PLASTIC or STATIC populations [stats]. We also observed that NON-PLASTIC lineages had significantly lower genotypic and phenotypic fidelity than either PLASTIC or STATIC lineages [stats]; that is, offspring along NON-PLASTIC lineages more frequently differed genotypically and phenotypically than offspring along either PLASTIC or STATIC lineages. Thus, our results indicate that NON-PLASTIC populations experienced both a greater magnitude of evolutionary change and a higher frequency of evolutionary change than either PLASTIC or STATIC populations.</p>
<p>We measured the fraction of mutated offspring along these lineages with a different phenotype than their parent. For lineages evolved in fluctuating environments, we evaluated mutants under both environmental conditions and counted all changes in the <em>aggregate</em> task profile. Consistent with previous results, we found that mutated offspring along NON-PLASTIC lineages exhibit more frequent phenotypic changes from their parent than mutated offspring from either PLASTIC or STATIC lineages [stats]. While mutational stability for PLASTIC and STATIC lineages is higher than for those from NON-PLASTIC lineages, we found that STATIC lineages exhibit a small, but significant, difference in mutational stability relative to PLASTIC lineages [stats]. Across all PLASTIC lineages, we found that over 80% ([data]) of changes to task profiles occur in phenotypic traits that are expressed only in the off environment (i.e., as cryptic variation). Approximately 52% ([data]) of this cryptic variation degraded tasks that would have been needed in the off environment, and the remaining <span class="math inline">∼</span>48% ([data]) were compensatory mutations; that is, they restored tasks before the environment changed.</p>
<p>In general, the evolution of adaptive plasticity stabilized PLASTIC treatment populations against environmental fluctuations, and their evolutionary dynamics more closely resembled those of populations evolving in a static environment. We observed no significant difference in coalescence events in PLASTIC and STATIC populations. We did, however, observe that, relative to STATIC populations, PLASTIC populations had more elapsed generations [significant, stats], mutation accumulation [stats], and phenotypic volatility [stats]; though these differences were statistically significant, they were not substantial.</p>
<p>[todo - genetic architecture results]</p>
<h2 id="the-evolution-of-novel-traits">The evolution of novel traits</h2>
<figure>
<embed src="media/complex-traits-panel.pdf" id="fig:complex-traits" style="width:90.0%" /><figcaption aria-hidden="true"> <strong>todo title.</strong> todo. </figcaption>
</figure>
<p>Adaptive phenotypic plasticity evolved in 42 of 100 replicates from the PLASTIC treatment during phase one of Experiment II; we used this more limited group to seed the 42 phase-two PLASTIC replicates. Figure [x] shows the task discovery, task performance, and task loss for PLASTIC, NON-PLASTIC, and STATIC treatments. We found that both PLASTIC and STATIC populations have significantly higher task performance at the end of the experiment than NON-PLASTIC populations [stats]. This, however, was not due to PLASTIC and STATIC lineages exploring a larger breadth of the fitness landscape, as NON-PLASTIC lineages had significantly higher task discovery than either PLASTIC or STATIC lineages [stats]. Indeed, we found that NON-PLASTIC lineages exhibited significantly more task loss than either PLASTIC or STATIC lineages [stats], as NON-PLASTIC lineages repeatedly failed to retain novel tasks.</p>
<p>As noted in our evolutionary change results, a larger number of generations elapsed in NON-PLASTIC populations than in PLASTIC or STATIC populations during our experiment. As such, we examined the <em>rates</em> of task loss and task discovery: are NON-PLASTIC lineages discovering and losing novel tasks more frequently than PLASTIC or STATIC lineages? We computed the per-generation rate of task discovery along dominant lineages and the per-generation rate of task loss along those same lineages. We found no significant difference in per-generation task discovery between NON-PLASTIC and STATIC lineages [stats], but we did find that PLASTIC lineages had a lower per-generation task discovery rate than NON-PLASTIC or STATIC lineages [stats]. We found that NON-PLASTIC lineages had a significantly higher per-generation rate of task loss than either PLASTIC or STATIC lineages [stats], while PLASTIC lineages had a small but significantly lower per-generation rate of task loss than STATIC lineages [stats]. We cannot reject the possibility that the larger magnitude of task discovery in NON-PLASTIC lineages was driven by a larger number of elapsed generations. Indeed, we found that NON-PLASTIC lineages repeatedly gained and failed to retain novel tasks during the experiment, and the final evolved genotypes performed fewer overall tasks.</p>
<p>Next, we examined the frequency at which novel task loss along lineages co-occurred with base task loss (i.e., loss of any of the original six tasks). Across all NON-PLASTIC dominant lineages, over 97% (10998 out of 11229) of instances of novel task loss co-occurred with a simultaneous change in base task profile. In contrast, across all PLASTIC and STATIC dominant lineages, we observed that approximately 20% (29 out of 142) and 2% (13 out of 631), respectively, of instances of novel task loss co-occurred with a simultaneous change in base task profile.</p>
<h2 id="deleterious-gene-accumulation">Deleterious gene accumulation</h2>
<p>Adaptive phenotypic plasticity evolved in 43 of 100 replicates from the PLASTIC treatment during phase one of Experiment III; we used this more limited group to seed the 43 phase-two PLASTIC replicates. At the end of our experiment, no dominant genotypes evolved in PLASTIC or STATIC treatments expressed the <code>poison</code> instruction under any environmental condition; however, the dominant genotype from 14 replicates of the NON-PLASTIC treatment did express the <code>poison</code> instruction at least once (statistically significant, [stats]). The <code>poison</code> execution in dominant organisms is expected to be kept minimal given that each execution is a 10% reduction in metabolic rate.</p>
<figure>
<embed src="media/poison-accumulation-panel.pdf" id="fig:deleterious-hitchhiking" style="width:100.0%" /><figcaption aria-hidden="true"> <strong>todo title.</strong> todo. </figcaption>
</figure>
<p>Figure [x] shows the [prevalence of poison instructions in final dominant genotypes and along dominant linages] for all treatments. We found that NON-PLASTIC lineages had significantly more organisms that executed the <code>poison</code> instruction more often than their parent as compared to PLASTIC or STATIC lineages [stats]; we found no significant difference in <code>poison</code> execution along dominant lineages between PLASTIC and STATIC lineages. These significance patterns do not qualitatively change when normalized by number of generations ([stats]).</p>
<p>Next, we examined the frequency that mutations that increase <code>poison</code> execution co-occur with mutations that change the base task profile of offspring along dominant lineages across treatments. Across all NON-PLASTIC dominant lineages, we found that approximately 49% (956 out of 1916) mutations that increased <code>poison</code> execution co-occurred with mutations that changed the base task profile; we observed zero instances of co-occurrence in PLASTIC and STATIC lineages. Additionally, we did not find compelling evidence that mutations that increased <code>poison</code> execution occurred as cryptic variation in PLASTIC lineages [supplement].</p>
<h1 id="discussion">Discussion</h1>
<p>Our results indicate that adaptive plasticity can improve evolution’s ability to maintain and refine novel traits, though with the tradeoff of reducing evolutionary exploration of the fitness landscape. These dynamics appear to be driven by the stabilizing effect that adaptive plasticity had on population dynamics rather than plasticity’s effect on genetic architecture or regulation. Below we discuss the specific implications of the results from each of our three experiments.</p>
<h2 id="the-evolution-of-adaptive-phenotypic-plasticity-slows-evolutionary-change-in-fluctuating-environments">The evolution of adaptive phenotypic plasticity slows evolutionary change in fluctuating environments</h2>
<p>Adaptively plastic populations experienced fewer selective sweeps and fewer total genetic changes relative to non-plastic populations evolving in the same environmental conditions. Plastic populations adapted to environmental changes by relying on environmentally-induced cues and within-lifetime regulation, which also stabilized these populations against fluctuations. Because ENV-A and ENV-B were incompatible (i.e., adaptive phenotypes in ENV-A were maladaptive in ENV-B and vice versa), non-plastic generalists did not evolve; any non-plastic generalist (e.g., that performed all tasks or performed no tasks) would have been quickly outcompeted by a specialist strategy because of the large fitness benefit for performing environment-specific tasks (and only those tasks). As such, non-plastic populations repeatedly required genetic modifications to respond to environmental changes and thus exhibited frequent selective sweeps during each new readaptation. Indeed, across all three of our experiments, the evolutionary dynamics of plastic populations were more similar to that of populations evolving in a static environment than to that of non-plastic populations evolving in an identical fluctuating environment.</p>
<p>This study is the first in-depth empirical investigation into how the <em>de novo</em> evolution of adaptive plasticity shifts the course of subsequent evolution in a cyclic environment. The evolutionary dynamics that we observed in non-plastic populations, however, are consistent with results from previous digital evolution studies. <span class="citation" data-cites="dolson_interpreting_2020">(Dolson et al. 2020)</span> proposed a suite of lineage and phylogeny metrics for quantifying the evolutionary histories of evolving populations of digital organisms; consistent with our findings, they observed that non-plastic populations that were evolved in cyclically changing environments exhibited higher phenotypic volatility and mutation accumulation than that of populations evolved in static conditions. <span class="citation" data-cites="lalejini_evolutionary_2016">(Lalejini and Ofria 2016)</span> visually inspected the evolutionary histories of non-plastic organisms evolved in fluctuating environments, observing that mutations readily switched the set of traits expressed by offspring. <span class="citation" data-cites="canino-koning_evolution_2016">(Canino-Koning, Wiser, and Ofria 2016)</span> investigated how different types of changing environments shape the genetic architectures of evolved organisms, showing that cyclically-changing environments can steer populations toward genotypes that more readily mutate to alternative phenotypes. <span class="citation" data-cites="canino-koning_evolution_2016">(Canino-Koning, Wiser, and Ofria 2016)</span> also observed that genomes evolved in harsh cyclic environments often contained vestigial fragments of genetic material adapted to prior environments.</p>
<p>Our results are also consistent with conventional evolutionary theory. A trait’s evolutionary response to selection depends on the strength of directional selection and on the amount of genetic variation for selection to act upon <span class="citation" data-cites="lande_measurement_1983 zimmer_evolution_2013">(Lande and Arnold 1983; Zimmer and Emlen 2013)</span>. In our experiments, non-plastic populations repeatedly experienced strong directional selection to toggle which tasks were expressed after each environmental change. As such, retrospective analyses of successful lineages revealed rapid evolutionary responses (that is, high rates of genetic and phenotypic changes). Evolved adaptive plasticity shielded populations from strong directional selection when the environment changed by eliminating the need for a rapid evolutionary response to toggle task expression. Indeed, both theoretical and empirical studies have shown that adaptive plasticity can constrain evolutionary change by weakening directional selection on evolving populations <span class="citation" data-cites="price_role_2003 paenke_influence_2007 ghalambor_non-adaptive_2015">(Price, Qvarnström, and Irwin 2003; Paenke, Sendhoff, and Kawecki 2007; Cameron K. Ghalambor et al. 2015)</span>.</p>
<p>We observed evidence of mutation accumulation in plastic lineages that resulted in the loss of unexpressed traits that would have been important for survival in the alternate environment. These mutations would have proven detrimental when the environment changed. However, because our analyses focused retrospectively on successful lineages, nearly all of these mutations were followed by a compensatory mutation that restored the lost trait before the environment changed and it needed to be expressed. We tuned the frequency of environmental fluctuations so that genes that needed to function appropriately in the off environment were able to remain in the population despite relaxed selection [See supplemental ???]. Additionally, we imposed no explicit costs on phenotypic plasticity, which minimized selection against the maintenance of plastic genes during the periods between environmental changes.</p>
<h2 id="adaptively-plastic-populations-retain-more-novel-traits-than-non-plastic-populations-in-fluctuating-environments">Adaptively plastic populations retain more novel traits than non-plastic populations in fluctuating environments</h2>
<p>In our second experiment, we evaluated if adaptive plasticity influences fitness landscape exploration and exploitation by testing the ability of different treatment populations to evolve and then maintain novel tasks. In fluctuating environments, we found that non-plastic populations explored more of the fitness landscape than adaptively plastic populations, as measured by the number of novel tasks discovered.</p>
<p>Despite lower overall task discovery, we found that adaptively plastic populations better exploited the fitness landscape, retaining a greater number of novel tasks than non-plastic populations evolving under identical environmental conditions. Evolution in non-plastic populations was dominated by numerous bouts of strong directional selection. This effect was driven by repeated environmental changes; after each change, the performance of the six base tasks needed to be realigned. In our experiment, novel tasks were less important to survival than the fluctuating base tasks. In non-plastic populations, mutations that improve an offspring’s fitness after an environmental change are enormously beneficial. As such beneficial mutations fix, they can carry with them co-occurring deleterious mutations that knock out novel tasks. Indeed, we found that mutations associated with novel task loss along successful lineages from non-plastic populations co-occurred with mutations that helped offspring adapt to environmental changes an overwhelming 97% of the time.</p>
<p><span class="citation" data-cites="nahum_improved_2017">(Nahum et al. 2017)</span> demonstrated that a single temporary environmental change can improve fitness landscape exploration and exploitation in evolving populations of non-plastic digital organisms. In our system, however, we found that <em>repeated</em> fluctuations reduced the ability of non-plastic populations to maintain and exploit tasks; that said, we did find that repeated fluctuations may improve overall task discovery. Consistent with our findings, <span class="citation" data-cites="canino-koning_fluctuating_2019">(Canino-Koning, Wiser, and Ofria 2019)</span> found that non-plastic populations of digital organisms evolving in a harsh cyclic environment maintained fewer novel traits than populations evolving in static environments.</p>
<p>Our results suggest that adaptive phenotypic plasticity can improve the potential for populations to exploit novel resources by stabilizing them against stressful environmental changes. The stability that we observe may also lend some support to the hypothesis that phenotypic plasticity can rescue populations from extinction under changing environmental conditions [cite].</p>
<p>Our data do not necessarily provide evidence for or against the genes as followers hypothesis. The genes as followers hypothesis focuses on contexts where plastic populations experience novel or abnormally stressful environmental change. However, in our system, environmental changes were cyclic (not novel), and the magnitude of changes were consistent for the entirety of the experiment (so none were abnormally stressful). Further, the introduction of novel tasks during the second phase of the experiment merely added additional static opportunities for fitness improvement and did not change the meaning of existing environmental cues.</p>
<h2 id="non-plastic-populations-experience-more-deleterious-gene-accumulation-in-fluctuating-environments">Non-plastic populations experience more deleterious gene accumulation in fluctuating environments</h2>
<p>In our third experiment, we evaluated if adaptive plasticity influences the accumulation of explicitly deleterious genes (<code>poison</code> instructions) in evolving genomes. We found that non-plastic lineages that evolved in a fluctuating environment exhibited both larger totals and higher rates of <code>poison</code> accumulation than that of adaptively plastic lineages. We did not find evidence of <code>poison</code> instructions accumulating as cryptic variation in plastic lineages. Indeed, <code>poison</code> accumulation in the lineages of adaptively plastic genotypes did not differ significantly from that of the lineages of genotypes evolved in static conditions.</p>
<p>We hypothesize that deleterious genetic hitchhiking drove <code>poison</code> instruction accumulation along non-plastic lineages in changing environments. In asexual populations without horizontal gene transfer, all co-occurring mutations are linked. As such, deleterious mutations linked with a stronger beneficial mutation (i.e., a driver) can sometimes ‘hitchhike’ to fixation <span class="citation" data-cites="smith_hitch-hiking_1974 van_den_bergh_experimental_2018 buskirk_hitchhiking_2017">(Smith and Haigh 1974; Van den Bergh et al. 2018; Buskirk, Peace, and Lang 2017)</span>. Natural selection normally prevents deleterious mutations from reaching high frequencies, as such mutants would be outcompeted. However, when a beneficial mutation sweeps to fixation in a clonal population, it carries along any linked genetic material, including other beneficial, neutral, or deleterious mutations <span class="citation" data-cites="barton_genetic_2000 smith_hitch-hiking_1974">(<strong>barton_genetic_2000?</strong>; Smith and Haigh 1974)</span>.</p>
<p>Across our experiments, the selective sweeps in non-plastic populations provided additional opportunities for genetic hitchhiking with each environmental change. Indeed, across our experiments, successful lineages from non-plastic populations in the cyclic environment exhibited higher mutation accumulation, novel trait loss, and <code>poison</code> instruction accumulation than their plastic counterparts. In aggregate, we found that many (<span class="math inline">∼</span>49%; XX / XX) mutations that increased <code>poison</code> instruction execution in offspring co-occurred with mutations that provided an even stronger benefit by adapting the offspring to an environmental change. This rate of co-occurrence is conservative because we did not analyze mutations that became linked in different generations.</p>
<p>We found that adaptive phenotypic plasticity reduced <code>poison</code> instruction accumulation by reducing the rate of evolutionary change, which in turn reduced opportunities for <code>poison</code> instructions to hitchhike to fixation. We did not find compelling evidence of cryptic variation harboring <code>poison</code> instructions in adaptively plastic lineages. We have two hypotheses for why we did not observe the accumulation of <code>poison</code> instructions in unexpressed plastic responses. First, the period of time between environmental changes was too fast for variants carrying unexpressed <code>poison</code> instructions to reach high frequencies before the environment changed, after which such variants would have been outcompeted. Second, the plastic machinery that evolves in Avida is typically well-integrated and highly specific; that is, plastic genomes usually adjust their phenotypic expression by toggling a minimal number of key instructions ([though, toggling of larger sequences is possible but more complex]) [cite - supplement]. As such, there is minimal genomic space for variation to accumulate in preexisting (but currently unexpressed) regulated regions.</p>
<h2 id="limitations-and-future-directions">Limitations and future directions</h2>
<p>Our conclusions are limited to <em>adaptively</em> plastic populations. We did not explore the effects of non-adaptive plasticity where environmental changes induced phenotypes that were further away from the local optimum. Non-adaptive plasticity can increase a population’s extinction risk, especially if the misaligned plastic response is strongly tied to survival or the population is not sufficiently large <span class="citation" data-cites="gomulkiewicz_when_1995 chevin_adaptation_2010">(Gomulkiewicz and Holt 1995; Chevin, Lande, and Mace 2010)</span>. If the population persists, however, non-adaptive plasticity has been shown to be capable of accelerating evolutionary change by increasing the strength of directional selection. <span class="citation" data-cites="ghalambor_non-adaptive_2015">(Cameron K. Ghalambor et al. 2015)</span>.</p>
<p>Environmental cues in our experiments were reliable, and environmental changes were consistent over time; that is, sensory instructions perfectly differentiated between ENV-A and ENV-B, and environmental fluctuations never exposed populations to entirely new conditions. Both the reliability of cues and the timescales of environment switching are known to influence evolutionary outcomes <span class="citation" data-cites="li_digital_2004 boyer_adaptation_2021">(Li and Wilke 2004; Boyer, Hérissant, and Sherlock 2021)</span> [cite - cue reliability]. For example, <span class="citation" data-cites="boyer_adaptation_2021">(Boyer, Hérissant, and Sherlock 2021)</span> evolved populations of <em>Saccharomyces cerevisiae</em> in an environment that fluctuated between two growth conditions, observing that both environmental predictability and switching rate influenced the rates of evolutionary responses as well as adaptive outcomes at the genotypic and phenotypic levels. In adaptively plastic populations, environmental switching rate can influence how plastic responses are maintained: long periods between switches can (but not always <span class="citation" data-cites="grant_maintenance_2020">(<strong>grant_maintenance_2020?</strong>)</span>) result in the loss of plasticity [citations]. [Given the flexibility of digital evolution techniques, next steps could extend our study to include different forms of plasticity (e.g., adaptive vs. non-adaptive), more complex environments with more than two possible states, stochastic environmental changes, and a range of environment switching rates.]</p>
<p>We focused our analyses on the lineages of final dominant genotypes. These successful lineages represented the majority of the evolutionary histories of populations at the end of our experiment, as populations did not exhibit long-term coexistence of different clades. Our analyses, therefore, gave us an accurate picture of [what fixed in the population]. We did not, however, examine the lineages of extinct clades. Future work will extend our analyses to include extinct lineages, giving us a more complete view of evolutionary history, which may allow us to better distinguish adaptively plastic populations from populations evolving in a static environment by measuring [variance in reproductive success within populations over time, ...].</p>
<p>As with most [evolution/wet-lab/??] experiments, our results are in the context of a particular model organism: “Avidian” self-replicating computer programs. Digital organisms in Avida regulate responses to environmental cues using a combination of sensory instructions and conditional logic instructions (<code>if</code> statements). The <code>if</code> instructions conditionally execute a single instruction depending on previous computations and the state of memory. As such, plastic genomes typically regulate a small number of key instructions that, when executed, change the expressed phenotype [supplement? or I’ll write a mini OSF document with old data to cite]. [This way of achieving plasticity more often results in small, independently regulated single-instruction sequences than in large, sequences of co-regulated instructions (which is possible but more complicated to encode).] This bias [toward high-specificity regulation] may limit the accumulation of hidden genetic variation in Avida genomes. However, as there are many model biological organisms, there are many model [computational] organisms (e.g., [citations]) that have different regulatory mechanisms that can be used to test the generality of our results. [A broad sentence to wrap everything up?]</p>
<h1 class="unnumbered" id="supplemental-data">Supplemental Data</h1>
<h1 class="unnumbered" id="data-availability-statement">Data Availability Statement</h1>
<h1 class="unnumbered" id="author-contributions">Author Contributions</h1>
<h1 class="unnumbered" id="funding">Funding</h1>
<h1 class="unnumbered" id="conflict-of-interest-statement">Conflict of Interest Statement</h1>
<p>The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.</p>
<h1 class="unnumbered" id="acknowledgments">Acknowledgments</h1>
<div id="refs" class="references csl-bib-body hanging-indent" role="doc-bibliography">
<div id="ref-ancel_undermining_2000" class="csl-entry" role="doc-biblioentry">
Ancel, Lauren W. 2000. <span>“Undermining the <span>Baldwin</span> <span>Expediting</span> <span>Effect</span>: <span>Does</span> <span>Phenotypic</span> <span>Plasticity</span> <span>Accelerate</span> <span>Evolution</span>?”</span> <em>Theoretical Population Biology</em> 58 (4): 307–19. <a href="https://doi.org/10.1006/tpbi.2000.1484">https://doi.org/10.1006/tpbi.2000.1484</a>.
</div>
<div id="ref-barrett_adaptation_2008" class="csl-entry" role="doc-biblioentry">
Barrett, R, and D Schluter. 2008. <span>“Adaptation from Standing Genetic Variation.”</span> <em>Trends in Ecology &amp; Evolution</em> 23 (1): 38–44. <a href="https://doi.org/10.1016/j.tree.2007.09.008">https://doi.org/10.1016/j.tree.2007.09.008</a>.
</div>
<div id="ref-beaumont_experimental_2009" class="csl-entry" role="doc-biblioentry">
Beaumont, Hubertus J. E., Jenna Gallie, Christian Kost, Gayle C. Ferguson, and Paul B. Rainey. 2009. <span>“Experimental Evolution of Bet Hedging.”</span> <em>Nature</em> 462 (7269): 90–93. <a href="https://doi.org/10.1038/nature08504">https://doi.org/10.1038/nature08504</a>.
</div>
<div id="ref-boyer_adaptation_2021" class="csl-entry" role="doc-biblioentry">
Boyer, Sébastien, Lucas Hérissant, and Gavin Sherlock. 2021. <span>“Adaptation Is Influenced by the Complexity of Environmental Change During Evolution in a Dynamic Environment.”</span> Edited by Geraldine Butler. <em>PLOS Genetics</em> 17 (1): e1009314. <a href="https://doi.org/10.1371/journal.pgen.1009314">https://doi.org/10.1371/journal.pgen.1009314</a>.
</div>
<div id="ref-buskirk_hitchhiking_2017" class="csl-entry" role="doc-biblioentry">
Buskirk, Sean W., Ryan Emily Peace, and Gregory I. Lang. 2017. <span>“Hitchhiking and Epistasis Give Rise to Cohort Dynamics in Adapting Populations.”</span> <em>Proceedings of the National Academy of Sciences</em> 114 (31): 8330–35. <a href="https://doi.org/10.1073/pnas.1702314114">https://doi.org/10.1073/pnas.1702314114</a>.
</div>
<div id="ref-canino-koning_evolution_2016" class="csl-entry" role="doc-biblioentry">
Canino-Koning, Rosangela, Michael J. Wiser, and Charles Ofria. 2016. <span>“The <span>Evolution</span> of <span>Evolvability</span>: <span>Changing</span> <span>Environments</span> <span>Promote</span> <span>Rapid</span> <span>Adaptation</span> in <span>Digital</span> <span>Organisms</span>.”</span> In <em>Proceedings of the <span>Artificial</span> <span>Life</span> <span>Conference</span> 2016</em>, 268–75. Cancun, Mexico: MIT Press. <a href="https://doi.org/10.7551/978-0-262-33936-0-ch047">https://doi.org/10.7551/978-0-262-33936-0-ch047</a>.
</div>
<div id="ref-canino-koning_fluctuating_2019" class="csl-entry" role="doc-biblioentry">
———. 2019. <span>“Fluctuating Environments Select for Short-Term Phenotypic Variation Leading to Long-Term Exploration.”</span> Edited by Laurent Keller. <em>PLOS Computational Biology</em> 15 (4): e1006445. <a href="https://doi.org/10.1371/journal.pcbi.1006445">https://doi.org/10.1371/journal.pcbi.1006445</a>.
</div>
<div id="ref-chadwick_parasite-mediated_2005" class="csl-entry" role="doc-biblioentry">
Chadwick, William, and Tom J Little. 2005. <span>“A Parasite-Mediated Life-History Shift in <em><span>Daphnia</span> Magna</em>.”</span> <em>Proceedings of the Royal Society B: Biological Sciences</em> 272 (1562): 505–9. <a href="https://doi.org/10.1098/rspb.2004.2959">https://doi.org/10.1098/rspb.2004.2959</a>.
</div>
<div id="ref-chevin_when_2010" class="csl-entry" role="doc-biblioentry">
Chevin, Luis-Miguel, and Russell Lande. 2010. <span>“When Do Adaptive Plasticity and Genetic Evolution Prevent Extinction of a Density‐regulated Population?”</span> <em>Evolution</em> 64 (4): 1143–50. <a href="https://doi.org/10.1111/j.1558-5646.2009.00875.x">https://doi.org/10.1111/j.1558-5646.2009.00875.x</a>.
</div>
<div id="ref-chevin_adaptation_2010" class="csl-entry" role="doc-biblioentry">
Chevin, Luis-Miguel, Russell Lande, and Georgina M. Mace. 2010. <span>“Adaptation, <span>Plasticity</span>, and <span>Extinction</span> in a <span>Changing</span> <span>Environment</span>: <span>Towards</span> a <span>Predictive</span> <span>Theory</span>.”</span> Edited by Joel G. Kingsolver. <em>PLoS Biology</em> 8 (4): e1000357. <a href="https://doi.org/10.1371/journal.pbio.1000357">https://doi.org/10.1371/journal.pbio.1000357</a>.
</div>
<div id="ref-clune_investigating_2007" class="csl-entry" role="doc-biblioentry">
Clune, Jeff, Charles Ofria, and Robert T. Pennock. 2007. <span>“Investigating the <span>Emergence</span> of <span>Phenotypic</span> <span>Plasticity</span> in <span>Evolving</span> <span>Digital</span> <span>Organisms</span>.”</span> In <em>Advances in <span>Artificial</span> <span>Life</span></em>, edited by Fernando Almeida e Costa, Luis Mateus Rocha, Ernesto Costa, Inman Harvey, and António Coutinho, 4648:74–83. Berlin, Heidelberg: Springer Berlin Heidelberg. <a href="https://doi.org/10.1007/978-3-540-74913-4_8">https://doi.org/10.1007/978-3-540-74913-4_8</a>.
</div>
<div id="ref-covert_experiments_2013" class="csl-entry" role="doc-biblioentry">
Covert, Arthur W., Richard E. Lenski, Claus O. Wilke, and Charles Ofria. 2013. <span>“Experiments on the Role of Deleterious Mutations as Stepping Stones in Adaptive Evolution.”</span> <em>Proceedings of the National Academy of Sciences</em> 110 (34): E3171–78. <a href="https://doi.org/10.1073/pnas.1313424110">https://doi.org/10.1073/pnas.1313424110</a>.
</div>
<div id="ref-crispo_baldwin_2007" class="csl-entry" role="doc-biblioentry">
Crispo, Erika. 2007. <span>“The <span>Baldwin</span> Effect and Genetic Assimilation: Revisiting Two Mechanisms of Evolutionary Change Mediated by Phenotypic Plasticity.”</span> <em>Evolution</em> 61 (11): 2469–79. <a href="https://doi.org/10.1111/j.1558-5646.2007.00203.x">https://doi.org/10.1111/j.1558-5646.2007.00203.x</a>.
</div>
<div id="ref-dolson_interpreting_2020" class="csl-entry" role="doc-biblioentry">
Dolson, Emily, Alexander Lalejini, Steven Jorgensen, and Charles Ofria. 2020. <span>“Interpreting the <span>Tape</span> of <span>Life</span>: <span>Ancestry</span>-<span>Based</span> <span>Analyses</span> <span>Provide</span> <span>Insights</span> and <span>Intuition</span> about <span>Evolutionary</span> <span>Dynamics</span>.”</span> <em>Artificial Life</em> 26 (1): 58–79. <a href="https://doi.org/10.1162/artl_a_00313">https://doi.org/10.1162/artl_a_00313</a>.
</div>
<div id="ref-dunn_two_2005" class="csl-entry" role="doc-biblioentry">
Dunn, Alison M., John C. Hogg, Andrew Kelly, and Melanie J. Hatcher. 2005. <span>“Two Cues for Sex Determination in <em><span>Gammarus</span> Duebeni</em> : <span>Adaptive</span> Variation in Environmental Sex Determination?”</span> <em>Limnology and Oceanography</em> 50 (1): 346–53. <a href="https://doi.org/10.4319/lo.2005.50.1.0346">https://doi.org/10.4319/lo.2005.50.1.0346</a>.
</div>
<div id="ref-forsman_rethinking_2015" class="csl-entry" role="doc-biblioentry">
Forsman, A. 2015. <span>“Rethinking Phenotypic Plasticity and Its Consequences for Individuals, Populations and Species.”</span> <em>Heredity</em> 115 (4): 276–84. <a href="https://doi.org/10.1038/hdy.2014.92">https://doi.org/10.1038/hdy.2014.92</a>.
</div>
<div id="ref-ghalambor_adaptive_2007" class="csl-entry" role="doc-biblioentry">
Ghalambor, C. K., J. K. McKAY, S. P. Carroll, and D. N. Reznick. 2007. <span>“Adaptive Versus Non-Adaptive Phenotypic Plasticity and the Potential for Contemporary Adaptation in New Environments.”</span> <em>Functional Ecology</em> 21 (3): 394–407. <a href="https://doi.org/10.1111/j.1365-2435.2007.01283.x">https://doi.org/10.1111/j.1365-2435.2007.01283.x</a>.
</div>
<div id="ref-ghalambor_behavior_2010" class="csl-entry" role="doc-biblioentry">
Ghalambor, Cameron K., Lisa M. Angeloni, and Scott P. Carroll. 2010. <span>“Behavior as Phenotypic Plasticity.”</span> In <em>Evolutionary Behavioral Ecology</em>, edited by David Westneat and Charles W. Fox, 90–107. New York, NY: Oxford University Press.
</div>
<div id="ref-ghalambor_non-adaptive_2015" class="csl-entry" role="doc-biblioentry">
Ghalambor, Cameron K., Kim L. Hoke, Emily W. Ruell, Eva K. Fischer, David N. Reznick, and Kimberly A. Hughes. 2015. <span>“Non-Adaptive Plasticity Potentiates Rapid Adaptive Evolution of Gene Expression in Nature.”</span> <em>Nature</em> 525 (7569): 372–75. <a href="https://doi.org/10.1038/nature15256">https://doi.org/10.1038/nature15256</a>.
</div>
<div id="ref-gibert_phenotypic_2019" class="csl-entry" role="doc-biblioentry">
Gibert, Patricia, Vincent Debat, and Cameron K Ghalambor. 2019. <span>“Phenotypic Plasticity, Global Change, and the Speed of Adaptive Evolution.”</span> <em>Current Opinion in Insect Science</em> 35 (October): 34–40. <a href="https://doi.org/10.1016/j.cois.2019.06.007">https://doi.org/10.1016/j.cois.2019.06.007</a>.
</div>
<div id="ref-gibson_uncovering_2004" class="csl-entry" role="doc-biblioentry">
Gibson, Greg, and Ian Dworkin. 2004. <span>“Uncovering Cryptic Genetic Variation.”</span> <em>Nature Reviews Genetics</em> 5 (9): 681–90. <a href="https://doi.org/10.1038/nrg1426">https://doi.org/10.1038/nrg1426</a>.
</div>
<div id="ref-goldsby_evolutionary_2014" class="csl-entry" role="doc-biblioentry">
Goldsby, Heather J., David B. Knoester, Charles Ofria, and Benjamin Kerr. 2014. <span>“The <span>Evolutionary</span> <span>Origin</span> of <span>Somatic</span> <span>Cells</span> Under the <span>Dirty</span> <span>Work</span> <span>Hypothesis</span>.”</span> Edited by Laurent Keller. <em>PLoS Biology</em> 12 (5): e1001858. <a href="https://doi.org/10.1371/journal.pbio.1001858">https://doi.org/10.1371/journal.pbio.1001858</a>.
</div>
<div id="ref-gomulkiewicz_when_1995" class="csl-entry" role="doc-biblioentry">
Gomulkiewicz, Richard, and Robert D. Holt. 1995. <span>“When Does <span>Evolution</span> by <span>Natural</span> <span>Selection</span> <span>Prevent</span> <span>Extinction</span>?”</span> <em>Evolution</em> 49 (1): 201. <a href="https://doi.org/10.2307/2410305">https://doi.org/10.2307/2410305</a>.
</div>
<div id="ref-gupta_study_1982" class="csl-entry" role="doc-biblioentry">
Gupta, Anand P., and R. C. Lewontin. 1982. <span>“A <span>Study</span> of <span>Reaction</span> <span>Norms</span> in <span>Natural</span> <span>Populations</span> of <span>Drosophila</span> Pseudoobscura.”</span> <em>Evolution</em> 36 (5): 934. <a href="https://doi.org/10.2307/2408073">https://doi.org/10.2307/2408073</a>.
</div>
<div id="ref-hendry_key_2016" class="csl-entry" role="doc-biblioentry">
Hendry, Andrew P. 2016. <span>“Key <span>Questions</span> on the <span>Role</span> of <span>Phenotypic</span> <span>Plasticity</span> in <span>Eco</span>-<span>Evolutionary</span> <span>Dynamics</span>.”</span> <em>Journal of Heredity</em> 107 (1): 25–41. <a href="https://doi.org/10.1093/jhered/esv060">https://doi.org/10.1093/jhered/esv060</a>.
</div>
<div id="ref-huey_behavioral_2003" class="csl-entry" role="doc-biblioentry">
Huey, Raymond B., Paul E. Hertz, and B. Sinervo. 2003. <span>“Behavioral <span>Drive</span> Versus <span>Behavioral</span> <span>Inertia</span> in <span>Evolution</span>: <span>A</span> <span>Null</span> <span>Model</span> <span>Approach</span>.”</span> <em>The American Naturalist</em> 161 (3): 357–66. <a href="https://doi.org/10.1086/346135">https://doi.org/10.1086/346135</a>.
</div>
<div id="ref-kruskal_use_1952" class="csl-entry" role="doc-biblioentry">
Kruskal, William H., and W. Allen Wallis. 1952. <span>“Use of <span>Ranks</span> in <span>One</span>-<span>Criterion</span> <span>Variance</span> <span>Analysis</span>.”</span> <em>Journal of the American Statistical Association</em> 47 (260): 583–621. <a href="https://doi.org/10.1080/01621459.1952.10483441">https://doi.org/10.1080/01621459.1952.10483441</a>.
</div>
<div id="ref-lalejini_evolutionary_2016" class="csl-entry" role="doc-biblioentry">
Lalejini, Alexander, and Charles Ofria. 2016. <span>“The <span>Evolutionary</span> <span>Origins</span> of <span>Phenotypic</span> <span>Plasticity</span>.”</span> In <em>Proceedings of the <span>Artificial</span> <span>Life</span> <span>Conference</span> 2016</em>, 372–79. Cancun, Mexico: MIT Press. <a href="https://doi.org/10.7551/978-0-262-33936-0-ch063">https://doi.org/10.7551/978-0-262-33936-0-ch063</a>.
</div>
<div id="ref-lande_measurement_1983" class="csl-entry" role="doc-biblioentry">
Lande, Russell, and Stevan J. Arnold. 1983. <span>“The <span>Measurement</span> of <span>Selection</span> on <span>Correlated</span> <span>Characters</span>.”</span> <em>Evolution</em> 37 (6): 1210. <a href="https://doi.org/10.2307/2408842">https://doi.org/10.2307/2408842</a>.
</div>
<div id="ref-lenski_genome_1999" class="csl-entry" role="doc-biblioentry">
Lenski, Richard E., Charles Ofria, Travis C. Collier, and Christoph Adami. 1999. <span>“Genome Complexity, Robustness and Genetic Interactions in Digital Organisms.”</span> <em>Nature</em> 400 (6745): 661–64. <a href="https://doi.org/10.1038/23245">https://doi.org/10.1038/23245</a>.
</div>
<div id="ref-lenski_evolutionary_2003" class="csl-entry" role="doc-biblioentry">
Lenski, Richard E., Charles Ofria, Robert T. Pennock, and Christoph Adami. 2003. <span>“The Evolutionary Origin of Complex Features.”</span> <em>Nature</em> 423 (6936): 139–44. <a href="https://doi.org/10.1038/nature01568">https://doi.org/10.1038/nature01568</a>.
</div>
<div id="ref-levis_evaluating_2016" class="csl-entry" role="doc-biblioentry">
Levis, Nicholas A., and David W. Pfennig. 2016. <span>“Evaluating <span>‘<span>Plasticity</span>-<span>First</span>’</span> <span>Evolution</span> in <span>Nature</span>: <span>Key</span> <span>Criteria</span> and <span>Empirical</span> <span>Approaches</span>.”</span> <em>Trends in Ecology &amp; Evolution</em> 31 (7): 563–74. <a href="https://doi.org/10.1016/j.tree.2016.03.012">https://doi.org/10.1016/j.tree.2016.03.012</a>.
</div>
<div id="ref-li_digital_2004" class="csl-entry" role="doc-biblioentry">
Li, Ye, and Claus O. Wilke. 2004. <span>“Digital <span>Evolution</span> in <span>Time</span>-<span>Dependent</span> <span>Fitness</span> <span>Landscapes</span>.”</span> <em>Artificial Life</em> 10 (2): 123–34. <a href="https://doi.org/10.1162/106454604773563559">https://doi.org/10.1162/106454604773563559</a>.
</div>
<div id="ref-mckinley_harnessing_2008" class="csl-entry" role="doc-biblioentry">
McKinley, P., B. H. C. Cheng, C. Ofria, D. Knoester, B. Beckmann, and H. Goldsby. 2008. <span>“Harnessing <span>Digital</span> <span>Evolution</span>.”</span> <em>Computer</em> 41 (1): 54–63. <a href="https://doi.org/10.1109/MC.2008.17">https://doi.org/10.1109/MC.2008.17</a>.
</div>
<div id="ref-nahum_improved_2017" class="csl-entry" role="doc-biblioentry">
Nahum, Joshua R., Jevin West, Benjamin M. Althouse, Luis Zaman, Charles Ofria, and Benjamin Kerr. 2017. <span>“Improved Adaptation in Exogenously and Endogenously Changing Environments.”</span> In <em>Proceedings of the 14th <span>European</span> <span>Conference</span> on <span>Artificial</span> <span>Life</span> <span>ECAL</span> 2017</em>, 306–13. Lyon, France: MIT Press. <a href="https://doi.org/10.7551/ecal_a_052">https://doi.org/10.7551/ecal_a_052</a>.
</div>
<div id="ref-ofria_avida:_2009" class="csl-entry" role="doc-biblioentry">
Ofria, Charles, David M. Bryson, and Claus O. Wilke. 2009. <span>“Avida: <span>A</span> <span>Software</span> <span>Platform</span> for <span>Research</span> in <span>Computational</span> <span>Evolutionary</span> <span>Biology</span>.”</span> In <em>Artificial <span>Life</span> <span>Models</span> in <span>Software</span></em>, edited by Maciej Komosinski and Andrew Adamatzky, 3–35. London: Springer London. <a href="https://doi.org/10.1007/978-1-84882-285-6_1">https://doi.org/10.1007/978-1-84882-285-6_1</a>.
</div>
<div id="ref-paaby_cryptic_2014" class="csl-entry" role="doc-biblioentry">
Paaby, Annalise B., and Matthew V. Rockman. 2014. <span>“Cryptic Genetic Variation: Evolution’s Hidden Substrate.”</span> <em>Nature Reviews Genetics</em> 15 (4): 247–58. <a href="https://doi.org/10.1038/nrg3688">https://doi.org/10.1038/nrg3688</a>.
</div>
<div id="ref-paenke_influence_2007" class="csl-entry" role="doc-biblioentry">
Paenke, Ingo, Bernhard Sendhoff, and Tadeusz J. Kawecki. 2007. <span>“Influence of <span>Plasticity</span> and <span>Learning</span> on <span>Evolution</span> Under <span>Directional</span> <span>Selection</span>.”</span> <em>The American Naturalist</em> 170 (2): E47–58. <a href="https://doi.org/10.1086/518952">https://doi.org/10.1086/518952</a>.
</div>
<div id="ref-pigliucci_phenotypic_2006" class="csl-entry" role="doc-biblioentry">
Pigliucci, M. 2006. <span>“Phenotypic Plasticity and Evolution by Genetic Assimilation.”</span> <em>Journal of Experimental Biology</em> 209 (12): 2362–67. <a href="https://doi.org/10.1242/jeb.02070">https://doi.org/10.1242/jeb.02070</a>.
</div>
<div id="ref-price_role_2003" class="csl-entry" role="doc-biblioentry">
Price, Trevor D., Anna Qvarnström, and Darren E. Irwin. 2003. <span>“The Role of Phenotypic Plasticity in Driving Genetic Evolution.”</span> <em>Proceedings of the Royal Society of London. Series B: Biological Sciences</em> 270 (1523): 1433–40. <a href="https://doi.org/10.1098/rspb.2003.2372">https://doi.org/10.1098/rspb.2003.2372</a>.
</div>
<div id="ref-rice_analyzing_1989" class="csl-entry" role="doc-biblioentry">
Rice, William R. 1989. <span>“Analyzing <span>Tables</span> of <span>Statistical</span> <span>Tests</span>.”</span> <em>Evolution</em> 43 (1): 223. <a href="https://doi.org/10.2307/2409177">https://doi.org/10.2307/2409177</a>.
</div>
<div id="ref-schaum_plasticity_2014" class="csl-entry" role="doc-biblioentry">
Schaum, C. Elisa, and Sinéad Collins. 2014. <span>“Plasticity Predicts Evolution in a Marine Alga.”</span> <em>Proceedings of the Royal Society B: Biological Sciences</em> 281 (1793): 20141486. <a href="https://doi.org/10.1098/rspb.2014.1486">https://doi.org/10.1098/rspb.2014.1486</a>.
</div>
<div id="ref-schlichting_hidden_2008" class="csl-entry" role="doc-biblioentry">
Schlichting, Carl D. 2008. <span>“Hidden <span>Reaction</span> <span>Norms</span>, <span>Cryptic</span> <span>Genetic</span> <span>Variation</span>, and <span>Evolvability</span>.”</span> <em>Annals of the New York Academy of Sciences</em> 1133 (1): 187–203. <a href="https://doi.org/10.1196/annals.1438.010">https://doi.org/10.1196/annals.1438.010</a>.
</div>
<div id="ref-schlichting_phenotypic_2014" class="csl-entry" role="doc-biblioentry">
Schlichting, Carl D., and Matthew A. Wund. 2014. <span>“Phenotypic <span>Plasticity</span> and <span>Epigenetic</span> <span>Marking</span>: <span>An</span> <span>Assessment</span> of <span>Evidence</span> for <span>Genetic</span> <span>Accommodation</span>.”</span> <em>Evolution</em> 68 (3): 656–72. <a href="https://doi.org/10.1111/evo.12348">https://doi.org/10.1111/evo.12348</a>.
</div>
<div id="ref-schwander_genes_2011" class="csl-entry" role="doc-biblioentry">
Schwander, Tanja, and Olof Leimar. 2011. <span>“Genes as Leaders and Followers in Evolution.”</span> <em>Trends in Ecology &amp; Evolution</em> 26 (3): 143–51. <a href="https://doi.org/10.1016/j.tree.2010.12.010">https://doi.org/10.1016/j.tree.2010.12.010</a>.
</div>
<div id="ref-smith_hitch-hiking_1974" class="csl-entry" role="doc-biblioentry">
Smith, John Maynard, and John Haigh. 1974. <span>“The Hitch-Hiking Effect of a Favourable Gene.”</span> <em>Genetical Research</em> 23 (1): 23–35. <a href="https://doi.org/10.1017/S0016672300014634">https://doi.org/10.1017/S0016672300014634</a>.
</div>
<div id="ref-van_den_bergh_experimental_2018" class="csl-entry" role="doc-biblioentry">
Van den Bergh, Bram, Toon Swings, Maarten Fauvart, and Jan Michiels. 2018. <span>“Experimental <span>Design</span>, <span>Population</span> <span>Dynamics</span>, and <span>Diversity</span> in <span>Microbial</span> <span>Experimental</span> <span>Evolution</span>.”</span> <em>Microbiology and Molecular Biology Reviews</em> 82 (3): e00008–18, /mmbr/82/3/e00008–18.atom. <a href="https://doi.org/10.1128/MMBR.00008-18">https://doi.org/10.1128/MMBR.00008-18</a>.
</div>
<div id="ref-wennersten_population-level_2012" class="csl-entry" role="doc-biblioentry">
Wennersten, Lena, and Anders Forsman. 2012. <span>“Population-Level Consequences of Polymorphism, Plasticity and Randomized Phenotype Switching: A Review of Predictions.”</span> <em>Biological Reviews</em> 87 (3): 756–67. <a href="https://doi.org/10.1111/j.1469-185X.2012.00231.x">https://doi.org/10.1111/j.1469-185X.2012.00231.x</a>.
</div>
<div id="ref-west-eberhard_developmental_2005" class="csl-entry" role="doc-biblioentry">
West-Eberhard, M. J. 2005. <span>“Developmental Plasticity and the Origin of Species Differences.”</span> <em>Proceedings of the National Academy of Sciences</em> 102 (Supplement 1): 6543–49. <a href="https://doi.org/10.1073/pnas.0501844102">https://doi.org/10.1073/pnas.0501844102</a>.
</div>
<div id="ref-west-eberhard_developmental_2003" class="csl-entry" role="doc-biblioentry">
West-Eberhard, May Jane. 2003. <em>Developmental <span>Plasticity</span> and <span>Evolution</span>.</em> Oxford University Press. <a href="http://search.ebscohost.com.proxy1.cl.msu.edu/login.aspx?direct=true&amp;db=e000xna&amp;AN=688592&amp;site=ehost-live">http://search.ebscohost.com.proxy1.cl.msu.edu/login.aspx?direct=true&amp;db=e000xna&amp;AN=688592&amp;site=ehost-live</a>.
</div>
<div id="ref-kotz_individual_1992" class="csl-entry" role="doc-biblioentry">
Wilcoxon, Frank. 1992. <span>“Individual <span>Comparisons</span> by <span>Ranking</span> <span>Methods</span>.”</span> In <em>Breakthroughs in <span>Statistics</span></em>, edited by Samuel Kotz and Norman L. Johnson, 196–202. New York, NY: Springer New York. <a href="https://doi.org/10.1007/978-1-4612-4380-9_16">https://doi.org/10.1007/978-1-4612-4380-9_16</a>.
</div>
<div id="ref-wilke_evolution_2001" class="csl-entry" role="doc-biblioentry">
Wilke, Claus O., Jia Lan Wang, Charles Ofria, Richard E. Lenski, and Christoph Adami. 2001. <span>“Evolution of Digital Organisms at High Mutation Rates Leads to Survival of the Flattest.”</span> <em>Nature</em> 412 (6844): 331–33. <a href="https://doi.org/10.1038/35085569">https://doi.org/10.1038/35085569</a>.
</div>
<div id="ref-winger_long_2019" class="csl-entry" role="doc-biblioentry">
Winger, Benjamin M., Giorgia G. Auteri, Teresa M. Pegan, and Brian C. Weeks. 2019. <span>“A Long Winter for the <span>Red</span> <span>Queen</span>: Rethinking the Evolution of Seasonal Migration.”</span> <em>Biological Reviews</em> 94 (3): 737–52. <a href="https://doi.org/10.1111/brv.12476">https://doi.org/10.1111/brv.12476</a>.
</div>
<div id="ref-wund_assessing_2012" class="csl-entry" role="doc-biblioentry">
Wund, M. A. 2012. <span>“Assessing the <span>Impacts</span> of <span>Phenotypic</span> <span>Plasticity</span> on <span>Evolution</span>.”</span> <em>Integrative and Comparative Biology</em> 52 (1): 5–15. <a href="https://doi.org/10.1093/icb/ics050">https://doi.org/10.1093/icb/ics050</a>.
</div>
<div id="ref-zheng_cryptic_2019" class="csl-entry" role="doc-biblioentry">
Zheng, Jia, Joshua L. Payne, and Andreas Wagner. 2019. <span>“Cryptic Genetic Variation Accelerates Evolution by Opening Access to Diverse Adaptive Peaks.”</span> <em>Science</em> 365 (6451): 347–53. <a href="https://doi.org/10.1126/science.aax1837">https://doi.org/10.1126/science.aax1837</a>.
</div>
<div id="ref-zimmer_evolution_2013" class="csl-entry" role="doc-biblioentry">
Zimmer, Carl, and Douglas John Emlen. 2013. <em>Evolution: Making Sense of Life</em>. Greenwood Village, CO: Roberts; Company Publishers.
</div>
</div>
<section class="footnotes" role="doc-endnotes">
<hr />
<ol>
<li id="fn1" role="doc-endnote"><p>We repeated our experiments without genome size restrictions and observed qualitatively similar results [cite supplement].<a href="#fnref1" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
<li id="fn2" role="doc-endnote"><p>One update in Avida is the amount of time required for the average organism to execute 30 instructions. See <span class="citation" data-cites="ofria_avida:_2009">(Ofria, Bryson, and Wilke 2009)</span> for more details.<a href="#fnref2" class="footnote-back" role="doc-backlink">↩︎</a></p></li>
</ol>
</section>
</body>
</html>
